#
# CDDL HEADER START
#
# The contents of this file are subject to the terms of the
# Common Development and Distribution License (the "License").
# You may not use this file except in compliance with the License.
#
# You can obtain a copy of the license at usr/src/OPENSOLARIS.LICENSE
# or http://www.opensolaris.org/os/licensing.
# See the License for the specific language governing permissions
# and limitations under the License.
#
# When distributing Covered Code, include this CDDL HEADER in each
# file and include the License file at usr/src/OPENSOLARIS.LICENSE.
# If applicable, add the following below this CDDL HEADER, with the
# fields enclosed by brackets "[]" replaced with your own identifying
# information: Portions Copyright [yyyy] [name of copyright owner]
#
# CDDL HEADER END
#
# Copyright 2009 Sun Microsystems, Inc.  All rights reserved.
# Use is subject to license terms.
#
#
#, fuzzy
msgid ""
msgstr "Project-Id-Version: PACKAGE VERSION\n"
       "Report-Msgid-Bugs-To: \n"
       "POT-Creation-Date: 2009-12-21 11:51+0900\n"
       "PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
       "Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
       "Language-Team: LANGUAGE <LL@li.org>\n"
       "MIME-Version: 1.0\n"
       "Content-Type: text/plain; charset=UTF-8\n"
       "Content-Transfer-Encoding: 8bit\n"
       "Plural-Forms: nplurals=INTEGER; plural=EXPRESSION;\n"

#. The full usage message isn't desired.
#: ../client.py:137
msgid "Try `pkg --help or -?' for more information."
msgstr "詳細は、pkg -- help または -? を参照してください。"

#: ../client.py:140
msgid ""
"Usage:\n"
"        pkg [options] command [cmd_options] [operands]\n"
"\n"
"Basic subcommands:\n"
"        pkg install [-nvq] [--no-refresh] [--no-index] package...\n"
"        pkg uninstall [-nrvq] [--no-index] package...\n"
"        pkg list [-Hafnsuv] [--no-refresh] [package...]\n"
"        pkg image-update [-fnvq] [--be-name name] [--no-refresh] [--no-"
"index]\n"
"        pkg refresh [--full] [publisher ...]\n"
"        pkg version\n"
"\n"
"Advanced subcommands:\n"
"        pkg info [-lr] [--license] [pkg_fmri_pattern ...]\n"
"        pkg search [-alprI] [-s server] query\n"
"        pkg verify [-Hqv] [pkg_fmri_pattern ...]\n"
"        pkg fix [pkg_fmri_pattern ...]\n"
"        pkg contents [-Hmr] [-a attribute=pattern ...] [-o attribute ...]\n"
"            [-s sort_key] [-t action_type ... ] [pkg_fmri_pattern ...]\n"
"        pkg image-create [-fFPUz] [--force] [--full|--partial|--user] [--"
"zone]\n"
"            [-k ssl_key] [-c ssl_cert] [--no-refresh]\n"
"            [--variant <variant_spec>=<instance>] \n"
"            [--facet <facet_spec>=[True,False]]\n"
"            (-p|--publisher) name=uri dir\n"
"        pkg change-variant [-nvq] [--be-name name] "
"<variant_spec>=<instance>\n"
"            [<variant_spec>=<instance> ...]\n"
"        pkg change-facet -nvq [--be-name name] <facet_spec>=[True|False|"
"None] ...\n"
"        pkg variant -H [<variant_spec>]\n"
"        pkg facet -H [<facet_spec>]\n"
"        pkg set-property propname propvalue\n"
"        pkg unset-property propname ...\n"
"        pkg property [-H] [propname ...]\n"
"\n"
"        pkg set-publisher [-Ped] [-k ssl_key] [-c ssl_cert]\n"
"            [-g origin_to_add | --add-origin=origin_to_add]\n"
"            [-G origin_to_remove | --remove-origin=origin_to_remove]\n"
"            [-m mirror_to_add | --add-mirror=mirror_to_add]\n"
"            [-M mirror_to_remove | --remove-mirror=mirror_to_remove]\n"
"            [--enable] [--disable] [--no-refresh] [--reset-uuid] \n"
"            [--non-sticky] [--sticky] [--search-after=publisher]\n"
"            [--search-before=publisher] publisher\n"
"        pkg unset-publisher publisher ...\n"
"        pkg publisher [-HPn] [publisher ...]\n"
"        pkg history [-Hl]\n"
"        pkg purge-history\n"
"        pkg rebuild-index\n"
"\n"
"Options:\n"
"        -R dir\n"
"        --help or -?\n"
"\n"
"Environment:\n"
"        PKG_IMAGE"
msgstr "使用方法:\n"
       "pkg [options] command [cmd_options] [operands]\n"
       "\n"
       "基本的なサブコマンド:\n"
       "pkg install [-nvq] [--no-refresh] [--no-index] package...\n"
       "pkg uninstall [-nrvq] [--no-index] package...\n"
       "pkg list [-Hafnsuv] [--no-refresh] [package...]\n"
       "pkg image-update [-fnvq] [--be-name name] [--no-refresh] [--no-"
       "index]\n"
       "pkg refresh [--full] [publisher ...]\n"
       "pkg version\n"
       "\n高度なサブコマンド:\n"
       "pkg info [-lr] [--license] [pkg_fmri_pattern ...]\n"
       "pkg search [-alprI] [-s server] query\n"
       "pkg verify [-Hqv] [pkg_fmri_pattern ...]\n"
       "pkg fix [pkg_fmri_pattern ...]\n"
       "pkg contents [-Hmr] [-a attribute=pattern ...] [-o attribute ...]\n"
       "[-s sort_key] [-t action_type ... ] [pkg_fmri_pattern ...]\n"
       "pkg image-create [-fFPUz] [--force] [--full|--partial|--user] [--"
       "zone]\n"
       "[-k ssl_key] [-c ssl_cert] [--no-refresh]\n"
       "[--variant <variant_spec>=<instance>] \n"
       "[--facet <facet_spec>=[True,False]]\n"
       "(-p|--publisher) name=uri dir\n"
       "pkg change-variant [-nvq] [--be-name name]"
       "<variant_spec>=<instance>\n"
       "[<variant_spec>=<instance> ...]\n"
       "pkg change-facet -nvq [--be-name name] <facet_spec>=[True|False|"
       "None] ...\n"
       "pkg variant -H [<variant_spec>]\n"
       "pkg facet -H [<facet_spec>]\n"
       "pkg set-property propname propvalue\n"
       "pkg unset-property propname ...\n"
       "pkg property [-H] [propname ...]\n"
       "\n"
       "pkg set-publisher [-Ped] [-k ssl_key] [-c ssl_cert]\n"
       "[-g origin_to_add | --add-origin=origin_to_add]\n"
       "[-G origin_to_remove | --remove-origin=origin_to_remove]\n"
       "[-m mirror_to_add | --add-mirror=mirror_to_add]\n"
       "[-M mirror_to_remove | --remove-mirror=mirror_to_remove]\n"
       "[--enable] [--disable] [--no-refresh] [--reset-uuid] \n"
       "[--non-sticky] [--sticky] [--search-after=publisher]\n"
       "[--search-before=publisher] publisher\n"
       "pkg unset-publisher publisher ...\n"
       "pkg publisher [-HPn] [publisher ...]\n"
       "pkg history [-Hl]\n"
       "pkg purge-history\n"
       "pkg rebuild-index\n"
       "\n"
       "オプション:\n"
       "-R dir\n"
       "--help or -?\n"
       "\n"
       "環境:\n"
       "PKG_IMAGE"

#: ../client.py:265
msgid "-f may only be used in combination with -a"
msgstr "-f は、-a と同時に指定した場合のみ使用できます"

#: ../client.py:275
#, python-format
msgid "%(opts)s may not be used with %(opt)s"
msgstr "%(opts)s と %(opts)s は一緒に使用できません"

#: ../client.py:279
msgid "-s and -v may not be combined"
msgstr "-s と -v を組み合わせることはできません"

#: ../client.py:377 ../client.py:433
msgid "installed"
msgstr "インストール済み"

#: ../client.py:379 ../client.py:429
msgid "known"
msgstr "既知"

#: ../client.py:399 ../client.py:1428 ../client.py:1826
msgid "no packages installed"
msgstr "インストール済みのパッケージが存在しません"

#: ../client.py:405
msgid "No specified packages have newer versions available."
msgstr "指定されたパッケージの中で、より新しいバージョンのあるものは存在しません"

#: ../client.py:408
msgid "No installed packages have newer versions available."
msgstr "インストールされたパッケージの中で、より新しいバージョンのあるものは存在しません"

#: ../client.py:431
msgid "known or installed"
msgstr "既知あるいはインストール済み"

#: ../client.py:435
#, python-format
msgid "no packages matching '%(pattern)s' %(state)s"
msgstr "'%(pattern)s’ の %(state)s に一致するパッケージが存在しません"

#: ../client.py:479
#, python-format
msgid "Created ZFS snapshot: %s"
msgstr "作成された ZFS スナップショット: %s"

#: ../client.py:486
msgid ""
"Requested \"fix\" operation would affect files that cannot be modified in "
"live image.\n"
"Please retry this operation on an alternate boot environment."
msgstr "要求された \"fix\" 操作は、ライブイメージ内で変更できないファイルに影響を及ぼします。\n 操作を代替起動環境で再試行してください。"

#: ../client.py:518 ../client.py:910 ../client.py:977 ../client.py:1037
msgid "-v and -q may not be combined"
msgstr "-v と -q を組み合わせることはできません"

#: ../client.py:571
msgid ""
"pkg: no packages matching the following patterns you specified are\n"
"installed on the system.\n"
msgstr ""
       "pkg: 指定された次のパターンに一致するパッケージの中で、\n"
       "システムにインストール済みのものは存在しません。\n"

#: ../client.py:581
#, python-format
msgid ""
"\n"
"All other patterns matched installed packages.  %s"
msgstr "\n ほかのパターンは、すべてインストール済みパッケージに一致しました。  %s"

#: ../client.py:607
#, python-format
msgid ""
"\n"
"An unexpected error happened while preparing for %s:"
msgstr "\n %s の準備中に予期しないエラーが発生しました:"

#: ../client.py:616
#, python-format
msgid "%s failed: %s"
msgstr "%s に失敗しました: %s"

#: ../client.py:619
#, python-format
msgid "%s cannot be done on live image"
msgstr "ライブイメージに %s を実行することはできません"

#: ../client.py:622
#, python-format
msgid ""
"Requested \"%s\" operation would affect files that cannot be modified in "
"live image.\n"
"Please retry this operation on an alternate boot environment."
msgstr "要求された \"%s\" 操作は、ライブイメージ内で変更できないファイルに影響を及ぼします。\n 操作を代替起動環境で再試行してください。"

#: ../client.py:654 ../client.py:658
#, python-format
msgid "An unexpected error happened during %s: %s"
msgstr "%s 中に予期しないエラーが発生しました: %s"

#: ../client.py:670 ../client.py:679
#, python-format
msgid "No image rooted at '%s'"
msgstr "'%s' をルートに持つイメージは存在しません"

#: ../client.py:682
#, python-format
msgid ""
"%s failed (inventory exception):\n"
"%s"
msgstr "%s が失敗しました (目録の例外):\n %s"

#: ../client.py:685
#, python-format
msgid ""
"WARNING: pkg(5) appears to be out of date, and should be updated before\n"
"running %s.  Please update pkg(5) using 'pfexec pkg install\n"
"SUNWipkg' and then retry the %s."
msgstr "警告: pkg(5) が古いようなので、%s を実行する前に\n 更新してください。'pfexec pkg install SUNWipkg'\n  を使って pkg(5) を更新したあとで、%s を再試行してください。"

#: ../client.py:692
#, python-format
msgid "Cannot remove '%s' due to the following packages that depend on it:"
msgstr "次のパッケージが依存しているため、'%s' を削除できません:"

#: ../client.py:742 ../client.py:814
#, python-format
msgid "%s: -v and -q may not be combined"
msgstr "%s: -v と -q を組み合わせることはできません"

#: ../client.py:745
#, python-format
msgid "%s: no variants specified"
msgstr "%s: バリアントが指定されていません"

#: ../client.py:751
#, python-format
msgid "%s: variants must to be of the form '<name>=<value>'."
msgstr "%s: バリアントは '<name>=<value>' の形式になっている必要があります。"

#: ../client.py:761
#, python-format
msgid "%s: duplicate variant specified: %s"
msgstr "%s: 重複したバリアントが指定されています: %s"

#: ../client.py:778 ../client.py:996
msgid "No updates necessary for this image."
msgstr "このイメージは更新の必要はありません。"

#: ../client.py:817
#, python-format
msgid "%s: no facets specified"
msgstr "%s: ファセットが指定されていません"

#: ../client.py:830
#, python-format
msgid "%s: facets must to be of the form 'facet....=[True|False|None]'"
msgstr "%s: ファセットは 'facet....=[True|False|None]' の形式になっている必要があります。"

#: ../client.py:839
#, python-format
msgid "%s: facets must to be of the form 'facet....=[True|False|None]'."
msgstr "%s: ファセットは 'facet....=[True|False|None]' の形式になっている必要があります。"

#: ../client.py:863
msgid "Facet change has no effect on image"
msgstr "ファセットの変更はイメージには影響しません"

#: ../client.py:913 ../client.py:2842
#, python-format
msgid "command does not take operands ('%s')"
msgstr "コマンドはオペランドを取りません ('%s')"

#: ../client.py:931
msgid "No updates available for this image."
msgstr "このイメージで使用可能な更新は存在しません。"

#: ../client.py:944
msgid "NOTE: Please review release notes posted at:\n"
msgstr "注: 次の場所で公開されているリリースノートを確認してください:\n"

#: ../client.py:974 ../client.py:1034
msgid "at least one package name required"
msgstr "パッケージ名を少なくとも 1 つ指定する必要があります"

#: ../client.py:1142
#, python-format
msgid ""
"The server returned a malformed result.\n"
"The problematic structure:%r"
msgstr "サーバーが不正な形式の結果を返しました。\n 問題のある構造: %r"

#: ../client.py:1148
#, python-format
msgid ""
"The server returned an invalid action.\n"
"%s"
msgstr "サーバーが不正なアクションを返しました。\n %s"

#: ../client.py:1204
#, python-format
msgid "%s is not a valid server URL."
msgstr "%s は有効なサーバー URL ではありません。"

#: ../client.py:1216
msgid "at least one search term must be provided"
msgstr "検索語句が少なくとも 1 つ指定されている必要があります"

#: ../client.py:1227
#, python-format
msgid "action level options ('%s') to -o cannot be used with the -p option"
msgstr "-o のアクションレベルのオプション ('%s') と -p オプションは同時に使用できません"

#: ../client.py:1279
#, python-format
msgid "The server returned a malformed result:%r"
msgstr "サーバーが不正な形式の結果を返しました:%r"

#: ../client.py:1290
msgid ""
"action level options to -o cannot be used with the queries that return "
"packages"
msgstr "-o のアクションレベルのオプションとパッケージを返すクエリーは同時に使用できません"

#: ../client.py:1347 ../client.py:1357
msgid ""
"The search index appears corrupted.  Please rebuild the index with 'pkg "
"rebuild-index'."
msgstr ""
       "検索インデックスが壊れているようです。'pkg "
       "rebuild-index' でインデックスを再構築してください。"

#: ../client.py:1391 ../client.py:1788
msgid "-l and -r may not be combined"
msgstr "-l と -r を組み合わせることはできません"

#: ../client.py:1394
msgid "must request remote info for specific packages"
msgstr "特定のパッケージではリモートの情報を要求する必要があります"

#: ../client.py:1446
msgid "Obsolete"
msgstr "廃止または互換性がなくなる可能性あり"

#: ../client.py:1448
msgid "Renamed"
msgstr "名前変更済み"

#: ../client.py:1456 ../packagemanager.py:1655 ../packagemanager.py:3119
msgid "Installed"
msgstr "インストール済み"

#: ../client.py:1458 ../packagemanager.py:3121
msgid "Not installed"
msgstr "インストール済みでない"

#: ../client.py:1460
msgid "          Name:"
msgstr "          名前:"

#: ../client.py:1462
msgid "       Summary:"
msgstr "                    概要:"

#: ../client.py:1464
msgid "   Description:"
msgstr "          説明:"

#: ../client.py:1474
msgid "      Category:"
msgstr "      カテゴリ:"

#: ../client.py:1481
msgid "         State:"
msgstr "          状態:"

#: ../client.py:1484
msgid "    Renamed to:"
msgstr "  変更後の名前:"

#. XXX even more info on the publisher would be nice?
#: ../client.py:1489
msgid "     Publisher:"
msgstr "        発行元:"

#: ../client.py:1490
msgid "       Version:"
msgstr "    バージョン:"

#: ../client.py:1491
msgid " Build Release:"
msgstr "ビルドリリース:"

#: ../client.py:1492
msgid "        Branch:"
msgstr "          分岐:"

#: ../client.py:1493
msgid "Packaging Date:"
msgstr "パッケージ化の日付:"

#: ../client.py:1494
msgid "          Size:"
msgstr "        サイズ:"

#: ../client.py:1495
msgid "          FMRI:"
msgstr "          FMRI:"

#: ../client.py:1503
msgid ""
"pkg: info: no packages matching the following patterns you specified are\n"
"installed on the system.  Try specifying -r to query remotely:"
msgstr "pkg: info: 指定された次のパターンに一致するパッケージの中で、\n システムにインストール済みのものが存在しません。-r を指定して、リモートでクエリーを実行してください:"

#: ../client.py:1507
msgid ""
"pkg: info: no packages matching the following patterns you specified were\n"
"found in the catalog.  Try relaxing the patterns, refreshing, and/or\n"
"examining the catalogs:"
msgstr "pkg: info: 指定された次のパターンに一致するパッケージが、カタログ内に\n 見つかりませんでした。パターンの条件緩和、再表示、カタログの検査のいずれか\n 、あるいはそのすべてを試してください:"

#: ../client.py:1523 ../modules/client/api_errors.py:198
#, python-format
msgid "'%s' matches multiple packages"
msgstr "'%s' は複数のパッケージに一致します"

#: ../client.py:1529
msgid "no license information could be found for the following packages:"
msgstr "次のパッケージのライセンス情報が見つかりませんでした:"

#: ../client.py:1745
#, python-format
msgid "Invalid attribute '%s'"
msgstr "無効な属性 '%s'"

#: ../client.py:1791
msgid "contents: must request remote contents for specific packages"
msgstr "contents: 特定のパッケージではリモートの内容を要求する必要があります"

#: ../client.py:1805
#, python-format
msgid "-m and %s may not be specified at the same time"
msgstr "-m と %s を同時に指定することはできません"

#: ../client.py:1868 ../client.py:1874
#, python-format
msgid "pkg: contents: '%s' matches multiple packages"
msgstr "pkg: contents: '%s' は複数のパッケージに一致します"

#: ../client.py:1954
msgid "pkg: contents: no matching actions found in the listed packages"
msgstr "pkg: contents: 一覧にあるパッケージの中には一致するアクションは見つかりませんでした"

#: ../client.py:1966
msgid ""
"pkg: contents: no packages matching the following patterns you specified "
"are\n"
"installed on the system.  Try specifying -r to query remotely:"
msgstr "pkg: contents: 指定された次のパターンに一致するパッケージの中で、\n システムにインストール済みのものが存在しません。-r を指定して、リモートで照会を行なってください:"

#: ../client.py:1970
msgid ""
"pkg: contents: no packages matching the following patterns you specified "
"were\n"
"found in the catalog.  Try relaxing the patterns, refreshing, and/or\n"
"examining the catalogs:"
msgstr "pkg: contents: 指定された次のパターンに一致するパッケージが、カタログ内に見つか\n りませんでした。パターンの条件緩和、再表示、カタログの検査のいずれか\n 、あるいはそのすべてを試してください:"

#: ../client.py:1987
#, python-format
msgid "pkg: %s/%s catalogs successfully updated:"
msgstr "pkg: %s/%s カタログの更新に成功しました:"

#: ../client.py:2055
msgid "'pkg publisher' will show a list of publishers."
msgstr "'pkg publisher' は発行元のリストを表示します。"

#: ../client.py:2132
msgid "requires a publisher name"
msgstr "発行元名が必要です"

#: ../client.py:2134
msgid "only one publisher name may be specified"
msgstr "指定できる発行元名は 1 つだけです"

#: ../client.py:2140
msgid "the -p and -d options may not be combined"
msgstr "-p オプションと -d オプションを組み合わせることはできません"

#: ../client.py:2144
msgid ""
"the -O and -g, --add-origin, -G, or --remove-origin options may not be "
"combined"
msgstr "-O オプションと -g、--add-origin、-G、 --remove-origin のいずれかのオプションを組み合わせることはできません"

#: ../client.py:2148
msgid "search_before and search_after may not be combined"
msgstr "search_before と search_after を組み合わせることはできません"

#: ../client.py:2167
msgid ""
"publisher does not exist. Use -g to define origin URI for new publisher."
msgstr "発行元が存在しません。 -g を使用して新しい発行元の起点 URI を定義してください。"

#: ../client.py:2271
#, python-format
msgid ""
"The origin URIs for '%(pubname)s' do not appear to point to a valid pkg "
"server.\n"
"Please check the server's address and client's network configuration.\n"
"Additional details:\n"
"\n"
"%(details)s"
msgstr "'%(pubname)s' の起点 URI は有効な pkg  サーバーを指していないようです。\n サーバーのアドレスとクライアントのネットワーク構成を確認してください。\n その他の詳細:\n \n %(details)s"

#: ../client.py:2299
msgid "at least one publisher must be specified"
msgstr "発行元が少なくとも 1 つ指定されている必要があります"

#: ../client.py:2329
#, python-format
msgid "Removal failed for '%(pub)s': %(msg)s"
msgstr "'%(pub)s' の削除に失敗しました: %(msg)s"

#: ../client.py:2397
msgid "PUBLISHER"
msgstr "発行元"

#: ../client.py:2397
msgid "TYPE"
msgstr "タイプ"

#: ../client.py:2397
msgid "STATUS"
msgstr "状態"

#: ../client.py:2398
msgid "URI"
msgstr "URI"

#: ../client.py:2423
msgid "non-sticky"
msgstr "非固定的"

#: ../client.py:2428
msgid "preferred"
msgstr "優先"

#: ../client.py:2430
msgid "disabled"
msgstr "無効"

#. XXX get the real origin status
#: ../client.py:2441
msgid "origin"
msgstr "起点"

#. XXX get the real mirror status
#: ../client.py:2445
msgid "mirror"
msgstr "ミラー"

#: ../client.py:2451
msgid "              SSL Key:"
msgstr "              SSL キー:"

#: ../client.py:2452
msgid "             SSL Cert:"
msgstr "             SSL 証明書:"

#: ../client.py:2464
msgid " Cert. Effective Date:"
msgstr " 証明書の発効日:"

#: ../client.py:2466
msgid "Cert. Expiration Date:"
msgstr "証明書の有効期限:"

#: ../client.py:2473
msgid "           Origin URI:"
msgstr "           起点 URI:"

#: ../client.py:2479
msgid "           Mirror URI:"
msgstr "           ミラー URI:"

#: ../client.py:2493
msgid "            Publisher:"
msgstr "    発行元:"

#: ../client.py:2494
msgid "                Alias:"
msgstr "                別名:"

#: ../client.py:2504
msgid "          Client UUID:"
msgstr "          クライアント UUID:"

#: ../client.py:2505
msgid "      Catalog Updated:"
msgstr "      カタログの更新:"

#: ../client.py:2507 ../client.py:2509
msgid "              Enabled:"
msgstr "              有効:"

#. Not Installed
#: ../client.py:2507 ../gui/modules/misc.py:213 ../gui/modules/misc.py:215
#: ../gui/modules/misc.py:273 ../gui/modules/misc.py:282
#: ../packagemanager.py:3523
msgid "No"
msgstr "いいえ"

#: ../client.py:2509
msgid "Yes"
msgstr "はい"

#: ../client.py:2520
msgid "requires a property name and value"
msgstr "プロパティー名と値が必要です"

#: ../client.py:2524 ../client.py:2552
msgid "set-publisher must be used to change the preferred publisher"
msgstr "優先発行元を変更するには set-publisher を使用する必要があります"

#: ../client.py:2547
msgid "requires at least one property name"
msgstr "プロパティー名を少なくとも 1 つ指定する必要があります"

#: ../client.py:2559
#, python-format
msgid "no such property: %s"
msgstr "そのようなプロパティーは存在しません: %s"

#: ../client.py:2581
#, python-format
msgid "property: no such property: %s"
msgstr "property: そのようなプロパティーは存在しません: %s"

#: ../client.py:2612
#, python-format
msgid "no such variant: %s"
msgstr "そのようなバリアントは存在しません: %s"

#: ../client.py:2699 ../client.py:2770
msgid "publisher argument must be of the form '<prefix>=<url>'."
msgstr "発行元の引数は '<prefix>=<url>' の形式になっている必要があります。"

#: ../client.py:2730
msgid "variant arguments must be of the form '<name>=<value>'."
msgstr "バリアントの引数は '<name>=<value>' の形式になっている必要があります。"

#: ../client.py:2740
msgid "Facet arguments must beform 'facet..=[True|False]'"
msgstr "ファセットの引数は 'facet..=[True|False]' の形式になっている必要があります"

#: ../client.py:2746
msgid "only one image directory path may be specified"
msgstr "指定できるイメージディレクトリのパスは 1 つだけです"

#: ../client.py:2767
msgid "a publisher must be specified"
msgstr "発行元を指定する必要があります"

#: ../client.py:2774
#, python-format
msgid "a publisher's prefix may not start with the text: %s"
msgstr "次のテキストで始まる発行元の接頭辞は使用できません: %s"

#: ../client.py:2779
msgid "publisher prefix contains invalid characters"
msgstr "発行元の接頭辞に無効な文字が含まれています"

#: ../client.py:2797
#, python-format
msgid "cannot create image at %(image_dir)s: %(reason)s"
msgstr "%(image_dir)s にイメージを作成できません: %(reason)s"

#: ../client.py:2814
#, python-format
msgid ""
"The URI '%(pub_url)s' does not appear to point to a valid pkg server.\n"
"Please check the server's address and client's network configuration.\n"
"Additional details:\n"
"\n"
"%(error)s"
msgstr "URI '%(pub_url)s' は有効な pkg  サーバーを指していないようです。\n"
       "サーバーのアドレスとクライアントのネットワーク構成を確認してください。\n"
       "その他の詳細:\n"
       "\n"
       "%(error)s"

#: ../client.py:2879
msgid "-H and -l may not be combined"
msgstr "-H と -l を組み合わせることはできません"

#: ../client.py:2883
msgid "TIME"
msgstr "時間"

#: ../client.py:2884
msgid "OPERATION"
msgstr "操作"

#: ../client.py:2884
msgid "CLIENT"
msgstr "クライアント"

#: ../client.py:2884
msgid "OUTCOME"
msgstr "結果"

#: ../client.py:2965
msgid "History purged."
msgstr "履歴が削除されます。"

#: ../client.py:2979
msgid ""
"\n"
"The following proxy configuration is set in the environment:\n"
msgstr "\n 次のプロキシ構成が環境に設定されました:\n"

#: ../client.py:2982
#, python-format
msgid "http_proxy: %s\n"
msgstr "http_proxy: %s\n"

#: ../client.py:2984
#, python-format
msgid "https_proxy: %s\n"
msgstr "https_proxy: %s\n"

#: ../client.py:2999 ../pkgdep.py:98 ../pkgdep.py:194 ../pkgdep.py:444
#, python-format
msgid "illegal global option -- %s"
msgstr "不正なグローバルオプション -- %s"

#: ../client.py:3011
#, python-format
msgid "%(opt)s takes argument of form name=value, not %(arg)s"
msgstr "%(opt)s は、%(arg)s ではなく、形式 name=value の引数を取ります"

#: ../client.py:3029
msgid "no subcommand specified"
msgstr "サブコマンドが指定されていません"

#: ../client.py:3040 ../client.py:3051
#, python-format
msgid "-R not allowed for %s subcommand"
msgstr "-R は %s サブコマンドでは許可されていません"

#: ../client.py:3047 ../client.py:3136
#, python-format
msgid "illegal option -- %s"
msgstr "不正なオプション -- %s"

#: ../client.py:3054
#, python-format
msgid "version: command does not take operands ('%s')"
msgstr "version: コマンドはオペランドを取りません ('%s')"

#: ../client.py:3079 ../pkgdep.py:236
msgid ""
"Could not find image.  Use the -R option or set $PKG_IMAGE to point\n"
"to an image, or change the working directory to one inside the image."
msgstr ""
       "イメージが見つかりませんでした。-R オプションを使用するか $PKG_IMAGE を設定してイメージの場所を指定するか、\n"
       "あるいは作業ディレクトリをイメージの内側に変更してください。"

#: ../client.py:3093
msgid "No image found."
msgstr "イメージが見つかりません。"

#: ../client.py:3129 ../pkgdep.py:472 ../publish.py:509
#, python-format
msgid "unknown subcommand '%s'"
msgstr "不明なサブコマンド '%s'"

#: ../client.py:3143 ../pkgdep.py:505 ../publish.py:535 ../pull.py:811
#, python-format
msgid ""
"\n"
"\n"
"This is an internal error.  Please let the developers know about this\n"
"problem by filing a bug at http://defect.opensolaris.org and including the\n"
"above traceback and this message.  The version of pkg(5) is '%s'."
msgstr ""
       "\n"
       "\n"
       "これは内部エラーです。この問題を開発者に知らせるために、\n"
       "http://defect.opensolaris.org でバグ情報を入力し、上記のトレースバック\n"
       "とこのメッセージを含めてください。pkg(5) のバージョンは '%s' です。"

#: ../client.py:3186
msgid ""
"\n"
"Errors were encountered while attempting to retrieve package or file data "
"for\n"
"the requested operation."
msgstr "\n 要求された操作に対するパッケージまたはファイルデータの取得を 試行しているときに、\n エラーが発生しました。"

#: ../client.py:3189
#, python-format
msgid ""
"Details follow:\n"
"\n"
"%s"
msgstr "詳細は次のとおりです:\n \n %s"

#: ../client.py:3195
msgid ""
"\n"
"Unable to contact a valid package depot. This may be due to a problem with "
"the server, network misconfiguration, or an incorrect pkg client "
"configuration.  Please check your network settings and attempt to contact "
"the server using a web browser."
msgstr ""
       "\n"
       "有効なパッケージデポに接続できません。原因としては、"
       "サーバーの問題、ネットワーク構成の誤り、pkg クライアント構成の"
       "誤りなどが考えられます。ネットワークの設定を確認し、Web ブラウザを使用して"
       "サーバーに接続してみてください。"

#: ../client.py:3200
#, python-format
msgid ""
"\n"
"Additional details:\n"
"\n"
"%s"
msgstr ""
       "\n"
       "その他の詳細:\n"
       "\n"
       "%s"

#: ../client.py:3208
msgid ""
"An error was encountered while attempting to load history information\n"
"about past client operations."
msgstr ""
       "以前のクライアント操作に関する履歴情報の読み込みを試行しているときに、\n"
       "エラーが発生しました。"

#: ../client.py:3217
msgid ""
"An error was encountered while attempting to store information about the\n"
"current operation in client history."
msgstr ""
       "現在の操作に関する情報をクライアント履歴に格納しようとしたときに、\n"
       "エラーが発生しました。"

#: ../client.py:3227
msgid "An error was encountered while attempting to purge client history."
msgstr "クライアント履歴の削除を試行しているときに、エラーが発生しました。"

#: ../client.py:3234
#, python-format
msgid ""
"The pkg command appears out of sync with the libraries provided \n"
"by SUNWipkg. The client version is %(client)s while the library API version "
"is %(api)s"
msgstr ""
       "pkg コマンドは、SUNWipkg で提供されているライブラリと同期がとれていない\n"
       "ようです。クライアントのバージョンが %(client)s であるのに対して、ライブラリ API "
       "のバージョンは %(api)s です。"

#: ../client.py:3249
#, python-format
msgid ""
"\n"
"%s%s"
msgstr "\n"
       "%s%s"

#: ../client.py:3251
msgid ""
"\n"
"\n"
"Despite the error while indexing, the image-update, install, or uninstall\n"
"has completed successfuly."
msgstr "\n"
       "\n"
       "インデックス作成、イメージ更新、インストール、またはアンインストール中にエラーが発生しましたが、\n"
       "正常に終了しました。"

#: ../depot.py:376 ../publish.py:115
msgid "property arguments must be of the form '<section.property>=<value>'."
msgstr "プロパティーの引数は '<section.property>=<value>' の形式になっている必要があります。"

#: ../gui/data/addmoresoftware.desktop.in.h:1
msgid "Add More Software"
msgstr "ソフトウェアの追加"

#: ../gui/data/addmoresoftware.desktop.in.h:2
#: ../gui/data/packagemanager.desktop.in.h:1
msgid "Install, update and remove software packages"
msgstr "ソフトウエアパッケージをインストール、更新または削除します"

#: ../gui/data/l10n.py:31
msgid "AMP Stack"
msgstr "AMP スタック"

#: ../gui/data/l10n.py:32
msgid "Accessories"
msgstr "アクセサリ"

#: ../gui/data/l10n.py:33
msgid "Administration and Configuration"
msgstr "管理と構成"

#: ../gui/data/l10n.py:34
msgid "Application and Web Servers"
msgstr "アプリケーションと Web サーバー"

#: ../gui/data/l10n.py:35
msgid "Builds"
msgstr "ビルド"

#: ../gui/data/l10n.py:36
msgid "C"
msgstr "C"

#: ../gui/data/l10n.py:37
msgid "C++"
msgstr "C++"

#: ../gui/data/l10n.py:38
msgid "Communications"
msgstr "通信"

#: ../gui/data/l10n.py:39
msgid "Configuration and Preferences"
msgstr "構成と設定"

#: ../gui/data/l10n.py:40
msgid "Core"
msgstr "コア"

#: ../gui/data/l10n.py:41
msgid "Databases"
msgstr "データベース"

#: ../gui/data/l10n.py:42
msgid "Databases (System)"
msgstr "データベース (システム)"

#: ../gui/data/l10n.py:43
msgid "Developer Tools"
msgstr "開発者ツール"

#: ../gui/data/l10n.py:44
msgid "Display"
msgstr "ディスプレイ"

#: ../gui/data/l10n.py:45
msgid "Distribution Tools"
msgstr "ディストリビューションツール"

#: ../gui/data/l10n.py:46
msgid "Documentation"
msgstr "マニュアル"

#: ../gui/data/l10n.py:47
msgid "Editors"
msgstr "エディタ"

#: ../gui/data/l10n.py:48
msgid "Enterprise Management"
msgstr "エンタープライズ管理"

#: ../gui/data/l10n.py:49
msgid "File Managers"
msgstr "ファイルマネージャー"

#: ../gui/data/l10n.py:50
msgid "File System"
msgstr "ファイルシステム"

#: ../gui/data/l10n.py:51
msgid "Fonts"
msgstr "フォント"

#: ../gui/data/l10n.py:52
msgid "GNOME and GTK+"
msgstr "GNOME と GTK+"

#: ../gui/data/l10n.py:53
msgid "GNU"
msgstr "GNU"

#: ../gui/data/l10n.py:54
msgid "Games"
msgstr "ゲーム"

#: ../gui/data/l10n.py:55
msgid "Graphics and Imaging"
msgstr "グラフィックスと画像処理"

#: ../gui/data/l10n.py:56
msgid "Hardware"
msgstr "ハードウェア"

#: ../gui/data/l10n.py:57
msgid "High Performance Computing"
msgstr "HPC"

#: ../gui/data/l10n.py:58
msgid "Integrated Development Environments"
msgstr "統合開発環境"

#: ../gui/data/l10n.py:59
msgid "Internationalization"
msgstr "国際化"

#: ../gui/data/l10n.py:60
msgid "Internet"
msgstr "インターネット"

#: ../gui/data/l10n.py:61
msgid "Java"
msgstr "Java"

#: ../gui/data/l10n.py:62
msgid "Libraries"
msgstr "ライブラリ"

#: ../gui/data/l10n.py:63
msgid "Libraries (System)"
msgstr "ライブラリ (システム)"

#: ../gui/data/l10n.py:64
msgid "Localizations (Desktop)"
msgstr "言語単位 (デスクトップ)"

#: ../gui/data/l10n.py:65
msgid "Localizations (System)"
msgstr "言語単位 (システム)"

#: ../gui/data/l10n.py:66
msgid "Media"
msgstr "メディア"

#: ../gui/data/l10n.py:67
msgid "Multimedia Libraries"
msgstr "マルチメディアライブラリ"

#: ../gui/data/l10n.py:68
msgid "Networking"
msgstr "ネットワーク処理"

#: ../gui/data/l10n.py:69
msgid "Office"
msgstr "オフィス"

#: ../gui/data/l10n.py:70
msgid "Office Tools"
msgstr "オフィスツール"

#: ../gui/data/l10n.py:71
msgid "Other Languages"
msgstr "その他のプログラミング言語"

#: ../gui/data/l10n.py:72
msgid "Other Peripherals"
msgstr "その他の周辺装置"

#: ../gui/data/l10n.py:73
msgid "PHP"
msgstr "PHP"

#: ../gui/data/l10n.py:74
msgid "Packaging"
msgstr "パッケージ処理"

#: ../gui/data/l10n.py:75
msgid "Panels and Applets"
msgstr "パネルとアプレット"

#: ../gui/data/l10n.py:76
msgid "Perl"
msgstr "Perl"

#: ../gui/data/l10n.py:77
msgid "Plug-ins and Run-times"
msgstr "プラグインとランタイム"

#: ../gui/data/l10n.py:78
msgid "Ports"
msgstr "ポート"

#: ../gui/data/l10n.py:79
msgid "Printing"
msgstr "印刷処理"

#: ../gui/data/l10n.py:80
msgid "Python"
msgstr "Python"

#: ../gui/data/l10n.py:81
msgid "Releases"
msgstr "リリース"

#: ../gui/data/l10n.py:82
msgid "Ruby"
msgstr "Ruby"

#: ../gui/data/l10n.py:83
msgid "Scripts"
msgstr "スクリプト"

#: ../gui/data/l10n.py:84
msgid "Security"
msgstr "セキュリティー"

#: ../gui/data/l10n.py:85
msgid "Services"
msgstr "サービス"

#: ../gui/data/l10n.py:86
msgid "Sessions"
msgstr "セッション"

#: ../gui/data/l10n.py:87
msgid "Shells"
msgstr "シェル"

#: ../gui/data/l10n.py:88
msgid "Software Management"
msgstr "ソフトウェア管理"

#: ../gui/data/l10n.py:89
msgid "Sound and Video"
msgstr "サウンドとビデオ"

#: ../gui/data/l10n.py:90
msgid "Source Code Management"
msgstr "ソースコード管理"

#: ../gui/data/l10n.py:91
msgid "Storage"
msgstr "ストレージ"

#: ../gui/data/l10n.py:92 ../packagemanager.py:1788
msgid "System"
msgstr "システム"

#: ../gui/data/l10n.py:93
msgid "System Utilities"
msgstr "システムユーティリティー"

#: ../gui/data/l10n.py:94
msgid "Text Tools"
msgstr "テキストツール"

#: ../gui/data/l10n.py:95
msgid "Theming"
msgstr "テーマ設定"

#: ../gui/data/l10n.py:96
msgid "Trusted"
msgstr "トラステッド"

#: ../gui/data/l10n.py:97
msgid "Trusted Extensions"
msgstr "Trusted Extensions"

#: ../gui/data/l10n.py:98
msgid "Universal Access"
msgstr "ユニバーサルアクセス"

#: ../gui/data/l10n.py:99
msgid "Virtualization"
msgstr "仮想化"

#: ../gui/data/l10n.py:100
msgid "Window Managers"
msgstr "ウィンドウマネージャー"

#: ../gui/data/l10n.py:101
msgid "X11"
msgstr "X11"

#: ../gui/data/l10n.py:102
msgid "X11 (System)"
msgstr "X11 (システム)"

#: ../gui/data/packagemanager-preferences.schemas.in.h:1
msgid ""
"A boolean which determines if the Start Page should be shown on startup or "
"the initial list view."
msgstr ""
       "起動時に「開始ページ」と初期リスト表示のどちらを表示するかを決定する"
       "ブール値。"

#: ../gui/data/packagemanager-preferences.schemas.in.h:2
msgid ""
"A boolean which determines if the search is executed dynamically as user "
"types or only when user hits the return key."
msgstr ""
       "ユーザーが入力するにつれて動的に検索を実行するか、ユーザーが"
       "Return キーを押したときだけ検索を実行するかを決定するブール値。"

#: ../gui/data/packagemanager-preferences.schemas.in.h:3
msgid ""
"A boolean which determines whether Package Manager should save its state on "
"exit and use it on restart. The state is the selected publisher and the size "
"and position of horizontal and vertical dividers."
msgstr "パッケージマネージャーに終了時の状態を保存し、次の起動時にその状態を使用するかを決定するブール値。保存される状態は、選択した発行元と、水平および垂直の境界バーの位置です。"

#: ../gui/data/packagemanager-preferences.schemas.in.h:4
msgid ""
"A boolean which determines whether Package Manager should show confirmation "
"dialog when the updates are found after performing Updates operation (Checks "
"if updates are available) or it should go straight to the update all "
"operation."
msgstr "パッケージマネージャーが更新操作 (更新があるかをチェック) 後に更新があった場合に確認ダイアログを表示するか、またはただちにすべての更新を行うかを決定するブール値。"

#: ../gui/data/packagemanager-preferences.schemas.in.h:5
msgid ""
"A boolean which determines whether Package Manager should show confirmation "
"dialog when the user selected some packages for installation or update."
msgstr "ユーザーがあるパッケージのインストールまたは更新を選択したときに、パッケージマネージャーが確認ダイアログを表示するかどうかを決定するブール値。"

#: ../gui/data/packagemanager-preferences.schemas.in.h:6
msgid ""
"A boolean which determines whether Package Manager should show confirmation "
"dialog when the user selected some packages for removal."
msgstr "ユーザーがあるパッケージの削除を選択したときに、パッケージマネージャーが確認ダイアログを表示するかどうかを決定するブール値。"

#: ../gui/data/packagemanager-preferences.schemas.in.h:7
msgid ""
"A boolean which determines whether Package Manager should start in search "
"mode."
msgstr "パッケージマネージャーを検索モードで開始するかを決定するブール値。"

#: ../gui/data/packagemanager-preferences.schemas.in.h:8
msgid ""
"A string which determines if the repository is shown or not in the error "
"dialog."
msgstr "エラーダイアログにリポジトリを表示するかどうかを決定する文字列。"

#: ../gui/data/packagemanager-preferences.schemas.in.h:9
msgid "A string which stores the last publisher used by package manager."
msgstr "パッケージマネージャーが最後に使用した発行元を保存する文字列。"

#: ../gui/data/packagemanager-preferences.schemas.in.h:10
msgid ""
"An integer which identifies the initial show filter value to be displayed "
"when packagemanager starts. 0 corresponds to All Packages."
msgstr "パッケージマネージャーの起動時に表示される表示フィルタの初期値を示す整数。 0 は「すべてのパッケージ」に対応します。"

#: ../gui/data/packagemanager-preferences.schemas.in.h:11
msgid ""
"An integer which identifies the initial toplevel section to be expanded when "
"packagemanager starts. 0 = \"All Categories\", 2 = \"Applications\" (the "
"default)"
msgstr "パッケージマネージャーの起動時に展開されるトップレベルセクションの初期値を示す整数。 0 は \"すべてのカテゴリ\" に、2 は \"アプリケーション\" (デフォルト) に対応します。"

#: ../gui/data/packagemanager-preferences.schemas.in.h:12
msgid ""
"An integer which specifies the initial height of the application main "
"window, overridden if there is not enough screen available."
msgstr ""
       "アプリケーションのメインウィンドウの初期の高さを指定する整数。"
       "スクリーンが十分でない場合は無効になります。"

#: ../gui/data/packagemanager-preferences.schemas.in.h:13
msgid ""
"An integer which specifies the initial horizonatal divider position of the "
"application, which separates the category panel from the list panel."
msgstr ""
       "アプリケーションの水平境界バーの初期位置を指定する整数。"
       "カテゴリパネルとリストパネルを区切ります。"

#: ../gui/data/packagemanager-preferences.schemas.in.h:14
msgid ""
"An integer which specifies the initial vertical divider position of the "
"application, which separates the details panel from the list panel."
msgstr ""
       "アプリケーションの垂直境界バーの初期位置を指定する整数。"
       "詳細パネルとリストパネルを区切ります。"

#: ../gui/data/packagemanager-preferences.schemas.in.h:15
msgid ""
"An integer which specifies the initial width of the application main window, "
"overridden if there is not enough screen available."
msgstr ""
       "アプリケーションのメインウィンドウの初期幅を指定する整数。"
       "スクリーンが十分でない場合は無効になります。"

#: ../gui/data/packagemanager-preferences.schemas.in.h:16
msgid ""
"An integer which specifies the maximum number of entries to store for search "
"completion."
msgstr "検索テキストの補完のために保存するエントリの最大件数を指定する整数。"

#: ../gui/data/packagemanager-preferences.schemas.in.h:17
msgid ""
"Comma separated list of repositories which we show if there are API errors "
"when searching them"
msgstr "リポジトリ検索時に API エラーが発生した場合に表示する、リポジトリをコンマ区切ったリスト"

#: ../gui/data/packagemanager-preferences.schemas.in.h:18
msgid "Display confirmation dialog after Check for updates"
msgstr "更新をチェックしたあとで確認ダイアログを表示する"

#: ../gui/data/packagemanager-preferences.schemas.in.h:19
msgid "Display install/update packages confirmation dialog"
msgstr "インストール/更新パッケージの確認ダイアログを表示する"

#: ../gui/data/packagemanager-preferences.schemas.in.h:20
msgid "Display remove packages confirmation dialog"
msgstr "削除パッケージの確認ダイアログを表示する"

#: ../gui/data/packagemanager-preferences.schemas.in.h:21
msgid "Display selected publisher at startup"
msgstr "起動時に選択した発行元を表示する"

#: ../gui/data/packagemanager-preferences.schemas.in.h:22
msgid "Enable type ahead search"
msgstr "先打ち検索を有効にする"

#: ../gui/data/packagemanager-preferences.schemas.in.h:23
msgid "Initial application height"
msgstr "アプリケーションの初期の高さ"

#: ../gui/data/packagemanager-preferences.schemas.in.h:24
msgid "Initial application horizonatal divider position"
msgstr "アプリケーションの水平境界バーの初期位置"

#: ../gui/data/packagemanager-preferences.schemas.in.h:25
msgid "Initial application vertical divider position"
msgstr "アプリケーションの垂直境界バーの初期位置"

#: ../gui/data/packagemanager-preferences.schemas.in.h:26
msgid "Initial application width"
msgstr "アプリケーションの初期幅"

#: ../gui/data/packagemanager-preferences.schemas.in.h:27
msgid "Initial section value"
msgstr "セクションの初期値"

#: ../gui/data/packagemanager-preferences.schemas.in.h:28
msgid "Initial show filter value"
msgstr "表示フィルタの初期値"

#: ../gui/data/packagemanager-preferences.schemas.in.h:29
msgid "Last Export Selections Path"
msgstr "最新のエクスポートパス"

#: ../gui/data/packagemanager-preferences.schemas.in.h:30
msgid "Last publisher used by package manager"
msgstr "パッケージマネージャーが最後に使用した発行元"

#: ../gui/data/packagemanager-preferences.schemas.in.h:31
msgid "Maximum entries for Search Completion"
msgstr "検索テキスト補完のための最大エントリ数"

#: ../gui/data/packagemanager-preferences.schemas.in.h:32
msgid "Show Start Page on startup"
msgstr "起動時に「開始ページ」を表示"

#: ../gui/data/packagemanager-preferences.schemas.in.h:33
msgid "Start Page Update URL"
msgstr "「開始ページ」の更新 URL"

#: ../gui/data/packagemanager-preferences.schemas.in.h:34
msgid "Start in serach mode"
msgstr "検索モードで開始"

#: ../gui/data/packagemanager-preferences.schemas.in.h:35
msgid ""
"The location from which to fetch updates to the Package Manager Start Page."
msgstr ""
       "パッケージマネージャーの「開始ページ」の更新を取得する場所。"

#: ../gui/data/packagemanager-preferences.schemas.in.h:36
msgid "The path for the last set of selections exported from packagemanager."
msgstr "パッケージマネージャーから最後にエクスポートされたパッケージのパス。"

#: ../gui/data/packagemanager.desktop.in.h:2
#: ../gui/data/packagemanager.glade.h:72 ../gui/modules/webinstall.py:428
#: ../packagemanager.py:277 ../packagemanager.py:3634
#: ../packagemanager.py:5097
msgid "Package Manager"
msgstr "パッケージマネージャー"

#: ../gui/data/packagemanager.glade.h:1
msgid "<b>Add, modify and remove publishers</b>"
msgstr "<b>発行元の追加、変更、および削除を行います</b>"

#: ../gui/data/packagemanager.glade.h:2
msgid "<b>Apply the following changes:</b>"
msgstr "<b>次の変更を適用します:</b>"

#: ../gui/data/packagemanager.glade.h:3
msgid "<b>Delete, rename and activate boot environments</b>"
msgstr "<b>ブート環境の削除、名前の変更、および有効化を行います。</b>"

#: ../gui/data/packagemanager.glade.h:4
msgid "<b>Exit</b>"
msgstr "<b>終了</b>"

#: ../gui/data/packagemanager.glade.h:5
msgid "<b>General</b>"
msgstr "<b>一般</b>"

#: ../gui/data/packagemanager.glade.h:6 ../packagemanager.py:3473
msgid "<b>Installable Version:</b>"
msgstr "<b>インストール可能なバージョン:</b>"

#: ../gui/data/packagemanager.glade.h:7
msgid "<b>Installed:</b>"
msgstr "<b>インストール済み:</b>"

#: ../gui/data/packagemanager.glade.h:8
msgid "<b>Name:</b>"
msgstr "<b>名前:</b>"

#: ../gui/data/packagemanager.glade.h:9
msgid "<b>Origins</b>"
msgstr "<b>起点</b>"

#: ../gui/data/packagemanager.glade.h:10
msgid "<b>Publisher added successfully</b>"
msgstr "<b>発行元は正常に追加されました</b>"

#: ../gui/data/packagemanager.glade.h:11
msgid "<b>Publisher</b>"
msgstr "<b>発行元</b>"

#: ../gui/data/packagemanager.glade.h:12
msgid "<b>Recent Searches</b>"
msgstr "<b>最近の検索</b>"

#: ../gui/data/packagemanager.glade.h:13
msgid "<b>Review the packages to be installed or udpated</b>"
msgstr "<b>インストールあるいは更新するパッケージを確認します</b>"

#: ../gui/data/packagemanager.glade.h:14
msgid "<b>Show Confirmation Dialog</b>"
msgstr "<b>確認のダイアログを表示</b>"

#: ../gui/data/packagemanager.glade.h:15
msgid "<b>Startup</b>"
msgstr "<b>起動</b>"

#: ../gui/data/packagemanager.glade.h:16
msgid "<b>Update All completed successfully</b>"
msgstr "<b>更新はすべて正常に完了しました</b>"

#: ../gui/data/packagemanager.glade.h:17
msgid "<b>_Mirrors</b>"
msgstr "<b>ミラー(_M)</b>"

#: ../gui/data/packagemanager.glade.h:18
msgid "<b>_SSL Key and Certificate</b>"
msgstr "<b>SSL キーと証明書(_S)</b>"

#: ../gui/data/packagemanager.glade.h:19
msgid "<big><b>Package Manager Web Installer</b></big>"
msgstr "<big><b>パッケージマネージャー Web インストーラ</b></big>"

#: ../gui/data/packagemanager.glade.h:20
msgid "<span foreground=\"blue\"><u>What's this?</u></span>"
msgstr "<span foreground=\"blue\"><u>これは何ですか ?</u></span>"

#: ../gui/data/packagemanager.glade.h:21
msgid ""
"A boot environment (BE) contains the operating\n"
"system image and updated packages. The\n"
"system will boot into the new BE on restart."
msgstr "ブート環境 (BE) は、オペレーティングシステムイメージと\n"
       "更新されたパッケージを含みます。\n"
       "システムは再起動時に新規ブート環境からブートします。"

#: ../gui/data/packagemanager.glade.h:24
msgid ""
"A new boot environment has been created and will be given the\n"
"following name."
msgstr "新規ブート環境が作成され\n"
       "それには次の名前がつけられます。"

#: ../gui/data/packagemanager.glade.h:26
msgid "A_dd"
msgstr "追加(_A)"

#: ../gui/data/packagemanager.glade.h:27
msgid "A_lias:"
msgstr "エイリアス(_L):"

#: ../gui/data/packagemanager.glade.h:28
msgid "About Glade"
msgstr "グレードについて"

#: ../gui/data/packagemanager.glade.h:29 ../gui/modules/repository.py:1327
msgid "Add Publisher"
msgstr "発行元を追加"

#: ../gui/data/packagemanager.glade.h:30
msgid "Adding Publisher Complete"
msgstr "発行元を追加しました"

#: ../gui/data/packagemanager.glade.h:31 ../gui/modules/repository.py:1430
#: ../gui/modules/repository.py:1444
msgid "Applying Changes"
msgstr "変更を適用する"

#: ../gui/data/packagemanager.glade.h:32
msgid "Applying changes..."
msgstr "変更を適用しています..."

#: ../gui/data/packagemanager.glade.h:33
msgid "B_rowse..."
msgstr "参照(_R)..."

#: ../gui/data/packagemanager.glade.h:34
msgid "Bro_wse..."
msgstr "参照(_W)..."

#: ../gui/data/packagemanager.glade.h:35
msgid "Browse for SSL Certificate"
msgstr "SSL 証明書を参照"

#: ../gui/data/packagemanager.glade.h:36
msgid "Browse for SSL Key"
msgstr "SSL キーを参照"

#: ../gui/data/packagemanager.glade.h:37
msgid "C_lear Search"
msgstr "検索を消去(_L)"

#: ../gui/data/packagemanager.glade.h:38
msgid "Cancel current operation"
msgstr "現在の操作をキャンセル"

#: ../gui/data/packagemanager.glade.h:39 ../um/data/updatemanager.glade.h:4
msgid "Checking for latest SUNWipkg and SUNWipkg-gui, please wait..."
msgstr "最新の SUNWipkg と SUNWipkg-gui をチェックしています。しばらくお待ちください..."

#: ../gui/data/packagemanager.glade.h:40
msgid "Checking package dependencies..."
msgstr "パッケージの依存関係をチェック中..."

#: ../gui/data/packagemanager.glade.h:41
msgid "Checks if updates are available"
msgstr "更新が使用可能かチェックします"

#: ../gui/data/packagemanager.glade.h:42
msgid "Clear Recent Searc_hes"
msgstr "最近の検索履歴を消去します(_H)"

#: ../gui/data/packagemanager.glade.h:43
msgid "Confirmation"
msgstr "確認"

#: ../gui/data/packagemanager.glade.h:44
msgid ""
"Copyright 2009 Sun Microsystems, Inc.  \n"
"All rights reserved.\n"
"Use is subject to license terms."
msgstr ""
       "Copyright 2009 Sun Microsystems, Inc.  \n"
       "All rights reserved.\n"
       "使用はライセンス契約条件に従うものとします。"

#: ../gui/data/packagemanager.glade.h:47
msgid "Currently active boot environment"
msgstr "現在アクティブになっているブート環境"

#: ../gui/data/packagemanager.glade.h:48
msgid "D_eselect All"
msgstr "すべてを選択解除(_E)"

#: ../gui/data/packagemanager.glade.h:49
msgid "D_etails"
msgstr "選択解除(_E)"

#: ../gui/data/packagemanager.glade.h:50
msgid "Dependencies"
msgstr "依存関係"

#: ../gui/data/packagemanager.glade.h:51 ../gui/modules/misc.py:185
msgid "Description:"
msgstr "説明:"

#: ../gui/data/packagemanager.glade.h:52
msgid "Details"
msgstr "詳細"

#: ../gui/data/packagemanager.glade.h:53
msgid "Do not notify me about these publishers again."
msgstr "今後、これらの発行元について通知しない。"

#: ../gui/data/packagemanager.glade.h:54 ../gui/modules/installupdate.py:92
#: ../um/data/updatemanager.glade.h:6
msgid "Download"
msgstr "ダウンロード"

#: ../gui/data/packagemanager.glade.h:55
msgid "Files"
msgstr "ファイル"

#: ../gui/data/packagemanager.glade.h:56
msgid "Go to package _list"
msgstr "package _list へ移動"

#: ../gui/data/packagemanager.glade.h:57 ../gui/modules/installupdate.py:93
msgid "Install"
msgstr "インストール"

#: ../gui/data/packagemanager.glade.h:58 ../gui/modules/installupdate.py:245
msgid "Install/Update"
msgstr "インストール/更新"

#: ../gui/data/packagemanager.glade.h:59
msgid "Install/Upgrade Check"
msgstr "インストール/更新のチェック"

#: ../gui/data/packagemanager.glade.h:60
msgid "License"
msgstr "ライセンス"

#: ../gui/data/packagemanager.glade.h:61
msgid "M_irror:"
msgstr "ミラー(_I):"

#: ../gui/data/packagemanager.glade.h:62
msgid "Manage Boot Environments"
msgstr "ブート環境を管理する"

#: ../gui/data/packagemanager.glade.h:63
msgid "Manage Publishers"
msgstr "発行元の管理"

#: ../gui/data/packagemanager.glade.h:64
msgid "Manage _Boot Environments..."
msgstr "ブート環境の管理(_B)..."

#: ../gui/data/packagemanager.glade.h:65
msgid "Modify Publisher"
msgstr "発行元の変更"

#: ../gui/data/packagemanager.glade.h:66 ../gui/modules/misc.py:184
msgid "Name:"
msgstr "名前:"

#: ../gui/data/packagemanager.glade.h:67
msgid "Name: "
msgstr "名前: "

#: ../gui/data/packagemanager.glade.h:68
msgid "O_rigin:"
msgstr "起点(_R):"

#: ../gui/data/packagemanager.glade.h:69
msgid "On Package _Install/Update"
msgstr "パッケージのインストール/更新時(_I)"

#: ../gui/data/packagemanager.glade.h:70
msgid "On Package _Remove"
msgstr "パッケージの削除時(_R)"

#: ../gui/data/packagemanager.glade.h:71
msgid "On _Updates"
msgstr "更新時(_U)"

#: ../gui/data/packagemanager.glade.h:73
msgid "Package Version Info"
msgstr "パッケージのバージョン情報"

#: ../gui/data/packagemanager.glade.h:74
msgid "Preferences"
msgstr "設定"

#: ../gui/data/packagemanager.glade.h:75 ../gui/modules/installupdate.py:91
msgid "Preparation"
msgstr "準備"

#: ../gui/data/packagemanager.glade.h:76 ../um/data/updatemanager.glade.h:9
msgid "Processing"
msgstr "処理中"

#: ../gui/data/packagemanager.glade.h:77
msgid "Pu_blisher:"
msgstr "発行元(_B):"

#: ../gui/data/packagemanager.glade.h:78
msgid "Publisher Name:"
msgstr "発行元名:"

#: ../gui/data/packagemanager.glade.h:79
msgid "Re_fresh"
msgstr "再表示(_F)"

#: ../gui/data/packagemanager.glade.h:80
msgid "Re_gistration Details"
msgstr "登録の詳細(_G)"

#: ../gui/data/packagemanager.glade.h:81
msgid "Read the release notes before restarting the system."
msgstr "システムを再起動する前にリリースノートをお読みください。"

#: ../gui/data/packagemanager.glade.h:82
msgid "Read the release notes."
msgstr "リリースノートをお読みください。"

#: ../gui/data/packagemanager.glade.h:83
msgid "Refresh"
msgstr "再表示"

#: ../gui/data/packagemanager.glade.h:84
msgid "Refresh list of packages and package status"
msgstr "パッケージとパッケージの状態一覧を再表示"

#: ../gui/data/packagemanager.glade.h:85
msgid "Release notes"
msgstr "リリースノート"

#: ../gui/data/packagemanager.glade.h:86
msgid "Remember current state on _exit"
msgstr "終了時に現在の状態を記憶する(_E)"

#: ../gui/data/packagemanager.glade.h:87
msgid "Remo_ve"
msgstr "削除(_V)"

#: ../gui/data/packagemanager.glade.h:88
msgid "Remote search error"
msgstr "リモート検索エラー"

#: ../gui/data/packagemanager.glade.h:89
msgid "Remov_e"
msgstr "削除(_E)"

#: ../gui/data/packagemanager.glade.h:90 ../gui/modules/installupdate.py:208
#: ../gui/modules/installupdate.py:211
msgid "Remove"
msgstr "削除"

#: ../gui/data/packagemanager.glade.h:91
msgid "Restart _Now"
msgstr "今すぐ再起動(_N)"

#: ../gui/data/packagemanager.glade.h:92
msgid "SSL C_ertificate: "
msgstr "SSL 証明書(_E): "

#: ../gui/data/packagemanager.glade.h:93
msgid "SSL Certi_ficate:"
msgstr "SSL 証明書(_F):"

#: ../gui/data/packagemanager.glade.h:94
msgid "SSL _Key:"
msgstr "SSL キー(_K):"

#: ../gui/data/packagemanager.glade.h:95
msgid "SSL _Key: "
msgstr "SSL キー(_K): "

#: ../gui/data/packagemanager.glade.h:96
msgid "Search"
msgstr "検索"

#: ../gui/data/packagemanager.glade.h:97
msgid "Search Publisher Errors"
msgstr "発行元エラーを検索"

#: ../gui/data/packagemanager.glade.h:98
msgid "Search Repositories Error"
msgstr "リポジトリエラーを検索"

#: ../gui/data/packagemanager.glade.h:99
msgid "Search errors with the following publishers:"
msgstr "次の発行元のエラーを検索:"

#: ../gui/data/packagemanager.glade.h:100
msgid "Select _Updates"
msgstr "更新を選択(_U)"

#: ../gui/data/packagemanager.glade.h:101
msgid "Select packages by marking the checkbox and click to Install/Update"
msgstr "チェックボックスにチェックマークを付けてパッケージを選択し、「インストール/更新」をクリックします"

#: ../gui/data/packagemanager.glade.h:102
msgid "Select packages by marking the checkbox and click to Remove selected"
msgstr "チェックボックスにチェックマークを付けてパッケージを選択し、「削除」をクリックします"

#: ../gui/data/packagemanager.glade.h:103
msgid "Set publisher priorities using arrows."
msgstr "矢印を使用して発行元のプロパティーを設定します。"

#: ../gui/data/packagemanager.glade.h:104
msgid "Show Start Page on _startup"
msgstr "起動時に「開始ページ」を表示(_S)"

#: ../gui/data/packagemanager.glade.h:105
msgid "Show _all publisher results"
msgstr "すべての発行元結果を表示(_A)"

#: ../gui/data/packagemanager.glade.h:106
msgid "Specify the publisher name and URI"
msgstr "発行元名と URI を指定"

#: ../gui/data/packagemanager.glade.h:107
msgid "Start _Page"
msgstr "開始ページ(_P)"

#: ../gui/data/packagemanager.glade.h:108
msgid "The following will be added to your system:"
msgstr "次のものがシステムに追加されます:"

#: ../gui/data/packagemanager.glade.h:109
msgid "This is a secure publisher."
msgstr "この発行元はセキュリティーで保護されています。 "

#: ../gui/data/packagemanager.glade.h:110
msgid ""
"This publisher requires an SSL key and an SSL certificate.\n"
"Specify the location of these files."
msgstr "この発行元には SSL キーと SSL 証明書が必要です。\n これらのファイルの場所を指定してください。"

#: ../gui/data/packagemanager.glade.h:112
msgid "Type text to search for the package."
msgstr "パッケージを検索するためのテキストを入力します。"

#: ../gui/data/packagemanager.glade.h:113
msgid "URI:"
msgstr "URI:"

#: ../gui/data/packagemanager.glade.h:114
msgid "Update All Complete"
msgstr "「すべて更新」が完了しました"

#: ../gui/data/packagemanager.glade.h:115 ../packagemanager.py:326
msgid "Updates"
msgstr "更新"

#: ../gui/data/packagemanager.glade.h:116
msgid "Vie_w:"
msgstr "表示(_W):"

#: ../gui/data/packagemanager.glade.h:117
msgid "_Add"
msgstr "追加(_A)"

#: ../gui/data/packagemanager.glade.h:118
msgid "_Add Publisher..."
msgstr "発行元を追加(_A)"

#: ../gui/data/packagemanager.glade.h:119
msgid "_Add..."
msgstr "追加(_A)..."

#: ../gui/data/packagemanager.glade.h:120
msgid "_Browse..."
msgstr "参照(_B)..."

#: ../gui/data/packagemanager.glade.h:121
msgid "_Contents"
msgstr "内容(_C)"

#: ../gui/data/packagemanager.glade.h:122 ../um/data/updatemanager.glade.h:13
msgid "_Details"
msgstr "詳細(_D)"

#: ../gui/data/packagemanager.glade.h:123
msgid "_Do not show this confirmation dialog again"
msgstr "今後、この確認ダイアログを表示しない(_D)"

#: ../gui/data/packagemanager.glade.h:124
msgid "_Down"
msgstr "下へ(_D)"

#: ../gui/data/packagemanager.glade.h:125
msgid "_Edit"
msgstr "編集(_E)"

#: ../gui/data/packagemanager.glade.h:126
msgid "_Export Selections..."
msgstr "選択項目をエクスポート(_E)..."

#: ../gui/data/packagemanager.glade.h:127
msgid "_File"
msgstr "ファイル(_F)"

#: ../gui/data/packagemanager.glade.h:128
msgid "_Help"
msgstr "ヘルプ(_H)"

#: ../gui/data/packagemanager.glade.h:129
msgid "_Install Details"
msgstr "インストールの詳細(_I)"

#: ../gui/data/packagemanager.glade.h:130
msgid "_Install/Update"
msgstr "インストール/更新(_I)"

#: ../gui/data/packagemanager.glade.h:131
msgid "_Manage Publishers..."
msgstr "発行元の管理(_M)..."

#: ../gui/data/packagemanager.glade.h:132
msgid "_Modify..."
msgstr "変更(_M)..."

#: ../gui/data/packagemanager.glade.h:133
msgid "_Name:"
msgstr "名前(_N): "

#: ../gui/data/packagemanager.glade.h:134
msgid "_Package"
msgstr "パッケージ(_P)"

#: ../gui/data/packagemanager.glade.h:135
msgid "_Package Version Info"
msgstr "パッケージのバージョン情報(_P)"

#: ../gui/data/packagemanager.glade.h:136
msgid "_Recent Searches"
msgstr "最新の検索結果(_R)"

#: ../gui/data/packagemanager.glade.h:137
msgid "_Registration Details"
msgstr "登録の詳細(_R)"

#: ../gui/data/packagemanager.glade.h:138
msgid "_Remove"
msgstr "削除(_R)"

#: ../gui/data/packagemanager.glade.h:139
msgid "_Remove Details"
msgstr "削除の詳細(_R)"

#: ../gui/data/packagemanager.glade.h:140
msgid "_Search"
msgstr "検索(_S)"

#: ../gui/data/packagemanager.glade.h:141
msgid "_URI:"
msgstr "URI(_U):"

#: ../gui/data/packagemanager.glade.h:142
msgid "_Up"
msgstr "上へ(_U)"

#: ../gui/data/packagemanager.glade.h:143
msgid "_Update Details"
msgstr "更新の詳細(_U)"

#: ../gui/data/packagemanager.glade.h:144
msgid "_Updates"
msgstr "更新(_U)"

#: ../gui/data/packagemanager.glade.h:145
msgid "_View"
msgstr "表示(_V)"

#: ../gui/data/packagemanager.glade.h:146 ../gui/modules/repository.py:75
msgid "e.g. http://pkg.opensolaris.org/release"
msgstr "例: http://pkg.opensolaris.org/release"

#: ../gui/data/packagemanager.glade.h:147
msgid "label"
msgstr "ラベル"

#: ../gui/modules/beadmin.py:73
msgid ""
"The <b>libbe</b> library was not found on your system.\n"
"All functions for managing Boot Environments are disabled"
msgstr ""
       "<b>libbe</b> ライブラリがシステム上に見つかりませんでした。\n"
       "ブート環境を管理するためのすべての機能が無効になっています"

#: ../gui/modules/beadmin.py:81
msgid "BE management"
msgstr "BE 管理"

#: ../gui/modules/beadmin.py:161
#, python-format
msgid "GUI will not respond to any event! %s. Check beadmin.py signals"
msgstr "GUI はどのイベントにも応答しません。%s。beadmin.py のシグナルをチェックしてください"

#: ../gui/modules/beadmin.py:170
msgid "Boot Environment Confirmation"
msgstr "ブート環境の確認"

#: ../gui/modules/beadmin.py:173
msgid "Loading Boot Environment Information"
msgstr "ブート環境情報の読み込み"

#: ../gui/modules/beadmin.py:175
msgid "Fetching BE entries..."
msgstr "BE エントリを取得しています..."

#: ../gui/modules/beadmin.py:202
msgid "Boot Environment"
msgstr "ブート環境"

#: ../gui/modules/beadmin.py:213
msgid "Created"
msgstr "作成済み"

#: ../gui/modules/beadmin.py:222
msgid "Size"
msgstr "サイズ"

#: ../gui/modules/beadmin.py:230
msgid "Active on Reboot"
msgstr "リブート時にアクティブ化"

#: ../gui/modules/beadmin.py:241
msgid "Delete"
msgstr "削除"

#: ../gui/modules/beadmin.py:256
msgid "Applying changes"
msgstr "変更を適用する"

#: ../gui/modules/beadmin.py:258
msgid "Applying changes, please wait ..."
msgstr "変更を適用しています。しばらくお待ちください..."

#: ../gui/modules/beadmin.py:287
msgid "Active on reboot\n"
msgstr "リブート時にアクティブ化\n"

#: ../gui/modules/beadmin.py:288
msgid "Delete\n"
msgstr "削除\n"

#: ../gui/modules/beadmin.py:289
msgid "Rename\n"
msgstr "名前の\n"

#: ../gui/modules/beadmin.py:330
msgid " to "
msgstr "変更"

#: ../gui/modules/beadmin.py:384
msgid "<b>Couldn't change Active Boot Environment to:</b>\n"
msgstr "<b>アクティブブート環境を次のものに変更できませんでした:</b>\n"

#: ../gui/modules/beadmin.py:389
msgid "<b>Couldn't delete Boot Environments:</b>\n"
msgstr "<b>ブート環境を削除できませんでした:</b>\n"

#: ../gui/modules/beadmin.py:396
msgid "<b>Couldn't rename Boot Environments:</b>\n"
msgstr "<b>ブート環境の名前を変更できませんでした:</b>\n"

#: ../gui/modules/beadmin.py:399
#, python-format
msgid "%s <b>to</b> %s\n"
msgstr "%s <b>変更後</b> %s\n"

#: ../gui/modules/beadmin.py:416
msgid "BE error"
msgstr "BE エラー"

#: ../gui/modules/beadmin.py:501
msgid ""
"The <b>libbe</b> library couldn't prepare list of Boot Environments.\n"
"All functions for managing Boot Environments are disabled"
msgstr ""
       "<b>libbe</b> ライブラリはブート環境の一覧を準備できませんでした。\n"
       "ブート環境を管理するためのすべての機能が無効になっています"

#: ../gui/modules/beadmin.py:531 ../gui/modules/beadmin.py:547
msgid "%m/%d/%y %H:%M"
msgstr "%Y 年 %m 月 %d 日 %H:%M"

#: ../gui/modules/installupdate.py:91
msgid "Preparing..."
msgstr "準備中..."

#: ../gui/modules/installupdate.py:92 ../gui/modules/installupdate.py:599
msgid "Downloading..."
msgstr "ダウンロード中..."

#: ../gui/modules/installupdate.py:93
msgid "Installing..."
msgstr "インストール中..."

#: ../gui/modules/installupdate.py:163 ../gui/modules/webinstall.py:216
msgid "_Proceed"
msgstr "続行(_P)"

#: ../gui/modules/installupdate.py:200
#, python-format
msgid "GUI will not respond to any event! %s. Check installupdate.py signals"
msgstr "GUI はどのイベントにも応答しません。%s。installupdate.py のシグナルをチェックしてください"

#. For the remove, we are not showing the download stage
#: ../gui/modules/installupdate.py:208
msgid "Removing..."
msgstr "削除中..."

#: ../gui/modules/installupdate.py:214
msgid "Remove Confirmation"
msgstr "削除の確認"

#: ../gui/modules/installupdate.py:217
msgid "Review the package to be removed"
msgid_plural "Review the packages to be removed"
msgstr[0] "削除するパッケージを確認します"
msgstr[1] "削除するパッケージを確認します"

#: ../gui/modules/installupdate.py:239
msgid "Update All"
msgstr "すべて更新"

#: ../gui/modules/installupdate.py:264
msgid "Install/Update Confirmation"
msgstr "インストール/更新の確認"

#: ../gui/modules/installupdate.py:266
msgid "Review the package to be Installed/Updated"
msgid_plural "Review the packages to be Installed/Updated"
msgstr[0] "インストール/更新するパッケージを確認します"
msgstr[1] "インストール/更新するパッケージを確認します"

#: ../gui/modules/installupdate.py:271
msgid "Update Confirmation"
msgstr "更新の確認"

#: ../gui/modules/installupdate.py:273
msgid "Review the package to be Updated"
msgid_plural "Review the packages to be Updated"
msgstr[0] "更新するパッケージを確認します"
msgstr[1] "更新するパッケージを確認します"

#: ../gui/modules/installupdate.py:278
msgid "Install Confirmation"
msgstr "インストールの確認"

#: ../gui/modules/installupdate.py:280
msgid "Review the package to be Installed"
msgid_plural "Review the packages to be Installed"
msgstr[0] "インストールするパッケージを確認します"
msgstr[1] "インストールするパッケージを確認します"

#: ../gui/modules/installupdate.py:315 ../packagemanager.py:1453
#: ../packagemanager.py:1561 ../updatemanager.py:297
msgid "Name"
msgstr "名前"

#: ../gui/modules/installupdate.py:322 ../gui/modules/repository.py:386
#: ../packagemanager.py:1473
msgid "Publisher"
msgstr "発行元"

#: ../gui/modules/installupdate.py:330 ../packagemanager.py:1489
msgid "Description"
msgstr "説明"

#: ../gui/modules/installupdate.py:375
msgid "Canceling..."
msgstr "キャンセル中..."

#: ../gui/modules/installupdate.py:413
#, python-format
msgid "Ensuring %s is up to date..."
msgstr "%s が最新であることを確認しています..."

#: ../gui/modules/installupdate.py:448
msgid "Inventory exception:\n"
msgstr "目録の例外:\n"

#: ../gui/modules/installupdate.py:457
msgid ""
"Please check the network connection.\n"
"Is the repository accessible?"
msgstr ""
       "ネットワーク接続を確認してください。\n"
       "リポジトリにアクセスできますか?"

#: ../gui/modules/installupdate.py:464
#, python-format
msgid ""
"Please check the network connection.\n"
"Is the repository accessible?\n"
"\n"
"%s"
msgstr ""
       "ネットワーク接続を確認してください。\n"
       "リポジトリにアクセスできますか?\n"
       "\n"
       "%s"

#: ../gui/modules/installupdate.py:470
#, python-format
msgid ""
"\n"
"Unable to contact a valid package depot. Please check your network\n"
"settings and attempt to contact the server using a web browser.\n"
"\n"
"%s"
msgstr "\n"
       "有効なパッケージデポに接続できません。ネットワークの設定を確認し、\n"
       "Web ブラウザを使用してサーバーに接続してみてください。\n"
       "\n"
       "%s"

#: ../gui/modules/installupdate.py:477
msgid ""
"pkg(5) appears to be out of date and should be updated.\n"
"Please update SUNWipkg package"
msgstr "pkg(5) は古くなっているため更新することをお勧めします。\n"
       "SUNWipkg パッケージを更新してください"

#: ../gui/modules/installupdate.py:483
#, python-format
msgid ""
"Cannot remove:\n"
"\t%s\n"
"Due to the following packages that depend on it:\n"
msgstr ""
       "削除できません:\n"
       "\t%s\n"
       "次のパッケージがこれに依存しているためです:\n"

#: ../gui/modules/installupdate.py:492
#, python-format
msgid ""
"\n"
"Failure of consistent use of pfexec or gksu when running\n"
"%s is often a source of this problem."
msgstr ""
       "\n"
       "多くの場合、%s の実行時に pfexec または gksu の使用方法を誤ったことが\n"
       "この問題の原因です。"

#: ../gui/modules/installupdate.py:495
msgid ""
"\n"
"To rebuild index, please use the terminal command:"
msgstr ""
       "\n"
       "インデックスを再構築するには、次の端末コマンドを使用してください:"

#: ../gui/modules/installupdate.py:496
msgid ""
"\n"
"\tpfexec pkg rebuild-index"
msgstr ""
       "\n"
       "\tpfexec pkg rebuild-index"

#: ../gui/modules/installupdate.py:500
msgid ""
"There was an error during installation. The search\n"
"index is corrupted. You might want try to fix this\n"
"problem by running command:\n"
"\tpfexec pkg rebuild-index"
msgstr ""
       "インストール中にエラーが発生しました。\n"
       "検索インデックスが壊れています。次のコマンドを実行することによって、\n"
       "この問題の解決を試みることをお勧めします:\n"
       "\tpfexec pkg rebuild-index"

#: ../gui/modules/installupdate.py:507
msgid ""
"This is an Live Image. The install\n"
"operation can't be performed."
msgstr ""
       "これはライブイメージです。インストール\n"
       "操作を実行できません。"

#: ../gui/modules/installupdate.py:512
msgid ""
"The requested operation would affect files that cannotbe modified in the "
"Live Image.\n"
"Please retry this operation on an alternate boot environment."
msgstr "要求された操作は、ライブイメージ内で変更できないファイルに影響を及ぼします。\n "
       "操作を代替起動環境で再試行してください。"

#: ../gui/modules/installupdate.py:518
#, python-format
msgid ""
"There was an error during installation.\n"
"The Plan of the operation is missing and the operation\n"
"can't be finished. You might want try to fix this\n"
"problem by restarting %s\n"
msgstr ""
       "インストール中にエラーが発生しました。\n"
       "操作の計画がないため\n"
       "操作を完了できません。%s を再起動することによって、\n"
       "この問題の解決を試みることをお勧めします\n"

#: ../gui/modules/installupdate.py:525
#, python-format
msgid ""
"There was an error during installation.\n"
"The State of the image is incorrect and the operation\n"
"can't be finished. You might want try to fix this\n"
"problem by restarting %s\n"
msgstr ""
       "インストール中にエラーが発生しました。\n"
       "イメージの状態が正しくないため\n"
       "操作を完了できません。%s を再起動することによって、\n"
       "この問題の解決を試みることをお勧めします\n"

#: ../gui/modules/installupdate.py:536
msgid "Specifying BE Name not supported.\n"
msgstr "BE 名を指定することはできません。\n"

#: ../gui/modules/installupdate.py:595
#, python-format
msgid "Updating %s"
msgstr "%s を更新しています"

#: ../gui/modules/installupdate.py:601
msgid "Executing..."
msgstr "実行しています..."

#: ../gui/modules/installupdate.py:607
msgid "Gathering package information, please wait..."
msgstr "パッケージ情報を収集しています。しばらくお待ちください..."

#: ../gui/modules/installupdate.py:609
msgid "Gathering package information"
msgstr "パッケージ情報を収集しています。"

#: ../gui/modules/installupdate.py:623
msgid "All packages already installed."
msgstr "すべてのパッケージがすでにインストールされています。"

#: ../gui/modules/installupdate.py:627
msgid ""
"Selected package(s) cannot be updated on their own.\n"
"Click Updates to update all packages."
msgstr "選択したパッケージは単独では更新できません。\n"
       "「更新」をクリックしてすべてのパッケージを更新してください。"

#: ../gui/modules/installupdate.py:631
msgid "No updates available"
msgstr "有効な更新はありません"

#: ../gui/modules/installupdate.py:692
msgid "Update All Confirmation"
msgstr "「すべて更新」の確認"

#: ../gui/modules/installupdate.py:695
msgid "Review the package which will be affected by Update all"
msgid_plural "Review the packages which will be affected by Update all"
msgstr[0] "「すべて更新」によって影響を受けるパッケージを確認します。"
msgstr[1] "「すべて更新」によって影響を受けるパッケージを確認します。"

#: ../gui/modules/installupdate.py:771 ../gui/modules/installupdate.py:811
msgid "No futher information available"
msgstr "利用できる詳細情報はありません"

#: ../gui/modules/installupdate.py:774 ../gui/modules/installupdate.py:802
msgid ""
"\n"
"Error:\n"
msgstr ""
       "\n"
       "エラー:\n"

#: ../gui/modules/installupdate.py:777 ../gui/modules/installupdate.py:793
msgid " - Failed </b>"
msgstr " - 失敗しました </b>"

#: ../gui/modules/installupdate.py:797
#, python-format
msgid ""
"An unknown error occurred in the %s stage.\n"
"Please let the developers know about this problem by\n"
"filing a bug together with the error details listed below at:\n"
msgstr "%s 段階で不明なエラーが発生しました。\n"
       "この問題を開発者に知らせるために、\n"
       "バグ情報と下記のエラーの詳細を次のサイトで入力してください:\n"

#: ../gui/modules/installupdate.py:806
msgid "Exception traceback:\n"
msgstr "例外のトレースバック:\n"

#: ../gui/modules/installupdate.py:813
msgid "pkg version: "
msgstr "pkg バージョン:"

#: ../gui/modules/installupdate.py:817
msgid "List of configured publishers:"
msgstr "構成済み発行元リスト:"

#: ../gui/modules/installupdate.py:920
msgid "Installation completed successfully"
msgstr "インストールが正常に完了しました"

#: ../gui/modules/installupdate.py:922
msgid "Packages removed successfully"
msgstr "パッケージが正常に削除されました"

#: ../gui/modules/installupdate.py:924
msgid "Packages updated successfully"
msgstr "パッケージが正常に更新されました"

#: ../gui/modules/installupdate.py:958
msgid ""
"Not enough disk space, the selected action cannot be performed.\n"
"\n"
"Click OK to manage your existing BEs and free up disk space or Cancel to "
"cancel the action."
msgstr ""
       "ディスク容量が不足しているため選択したアクションを実行できません。\n"
       "\n"
       "既存の BE を管理してディスク容量を解放する場合は「了解」をクリックし、アクションを取り消す場合は"
       "「取消し」をクリックしてください。"

#: ../gui/modules/installupdate.py:962
msgid "Not Enough Disk Space"
msgstr "ディスク容量の不足"

#: ../gui/modules/installupdate.py:980
msgid "Packages To Be Updated:\n"
msgstr "更新するパッケージ:\n"

#: ../gui/modules/installupdate.py:987
msgid "Packages To Be Installed:\n"
msgstr "インストールするパッケージ:\n"

#: ../gui/modules/installupdate.py:994
msgid "Packages To Be Removed:\n"
msgstr "削除するパッケージ:\n"

#: ../gui/modules/misc.py:135
#, python-format
msgid "Version mismatch: expected version %d, got version %d"
msgstr "バージョンの不一致: 必要なバージョン %d: 取得したバージョン %d"

#: ../gui/modules/misc.py:138
#, python-format
msgid "%s is not an install image"
msgstr "%s はインストールイメージではありません"

#: ../gui/modules/misc.py:142
msgid "API Error"
msgstr "API エラー"

#: ../gui/modules/misc.py:163
msgid "Error"
msgstr "エラー"

#: ../gui/modules/misc.py:186
msgid "Size:"
msgstr "サイズ:"

#: ../gui/modules/misc.py:187
msgid "Category:"
msgstr "カテゴリ:"

#: ../gui/modules/misc.py:188
msgid "Installed:"
msgstr "インストール済み:"

#: ../gui/modules/misc.py:189
msgid "Version Available:"
msgstr "使用可能なバージョン:"

#: ../gui/modules/misc.py:190 ../gui/modules/misc.py:206
#: ../gui/modules/misc.py:216
msgid "Latest Version:"
msgstr "最新バージョン:"

#: ../gui/modules/misc.py:191
msgid "Publisher:"
msgstr "発行元:"

#. Installed with later version but can't upgrade
#. Upgradeable Version: None
#. Not Installed with later version and can't install
#. Installable Version: None
#: ../gui/modules/misc.py:193 ../gui/modules/misc.py:226
#: ../packagemanager.py:3492 ../packagemanager.py:3550
#: ../packagemanager.py:4128
msgid "None"
msgstr "なし"

#: ../gui/modules/misc.py:201 ../packagemanager.py:3478
#, python-format
msgid "Yes, %(version)s (Build %(build)s-%(branch)s)"
msgstr "はい、 %(version)s (ビルド %(build)s-%(branch)s)"

#: ../gui/modules/misc.py:218 ../packagemanager.py:3470
#: ../packagemanager.py:3566 ../packagemanager.py:4096
#, python-format
msgid "%(version)s (Build %(build)s-%(branch)s)"
msgstr "%(version)s (ビルド %(build)s-%(branch)s)"

#: ../gui/modules/misc.py:243
msgid " (disabled)"
msgstr "(無効)"

#: ../gui/modules/misc.py:245
msgid " (removed)"
msgstr "(削除済み)"

#: ../gui/modules/misc.py:366
msgid "Catalog refresh error:\n"
msgstr "カタログ更新エラー:\n"

#: ../gui/modules/misc.py:369
#, python-format
msgid "Only %(suc)s out of %(tot)s catalogs successfully updated.\n"
msgstr "%(tot)s 個のうち、%(suc)s 個のカタログだけ正常に更新されました。\n"

#: ../gui/modules/progress.py:44
#, python-format
msgid "Retrieving catalog '%s'...\n"
msgstr "カタログ '%s' を取得しています...\n"

#: ../gui/modules/progress.py:52
msgid "Caching catalogs ...\n"
msgstr "カタログをキャッシュしています...\n"

#: ../gui/modules/progress.py:59
msgid "Loading catalog cache ...\n"
msgstr "カタログキャッシュを読み込んでいます...\n"

#: ../gui/modules/progress.py:70
#, python-format
msgid "Refreshing catalog %s\n"
msgstr "カタログ %s を再表示しています\n"

#: ../gui/modules/progress.py:75
#, python-format
msgid "Finished refreshing catalog %s\n"
msgstr "カタログ %s の再表示を終了しました\n"

#: ../gui/modules/progress.py:87
#, python-format
msgid "Evaluating: %s"
msgstr "評価中: %s"

#: ../gui/modules/progress.py:110
#, python-format
msgid "Downloaded %(current)s of %(total)s"
msgstr "%(current)s/%(total)s がダウンロードされました"

#: ../gui/modules/progress.py:117
#, python-format
msgid "Package %d of %d: %s\n"
msgstr "パッケージ %d/%d: %s\n"

#: ../gui/modules/progress.py:127 ../gui/modules/progress.py:139
#, python-format
msgid "%s\n"
msgstr "%s\n"

#: ../gui/modules/repository.py:103
msgid "Manage Publishers Confirmation"
msgstr "発行元確認の管理"

#: ../gui/modules/repository.py:339
#, python-format
msgid "GUI will not respond to any event! %s. Check repository.py signals"
msgstr "GUI はどのイベントにも応答しません。%s。repository.py のシグナルをチェックしてください"

#: ../gui/modules/repository.py:393
msgid "Alias"
msgstr "別名"

#: ../gui/modules/repository.py:399
msgid "Enabled"
msgstr "有効"

#: ../gui/modules/repository.py:528
msgid "Name contains invalid characters"
msgstr "名前に無効な文字が含まれています"

#: ../gui/modules/repository.py:531
msgid "Name already in use"
msgstr "名前はすでに使用されています"

#: ../gui/modules/repository.py:569 ../gui/modules/repository.py:1669
msgid "SSL Key not found at specified location"
msgstr "指定された場所には SSL 鍵が見つかりません"

#: ../gui/modules/repository.py:778
msgid "Adding Publisher"
msgstr "発行元の追加"

#: ../gui/modules/repository.py:792
msgid "Canceling...\n"
msgstr "キャンセルしています...\n"

#: ../gui/modules/repository.py:813 ../gui/modules/repository.py:1028
#, python-format
msgid "Removing publisher %s\n"
msgstr "発行元 %s を削除しています\n"

#: ../gui/modules/repository.py:817 ../gui/modules/repository.py:1032
#, python-format
msgid "Publisher %s succesfully removed\n"
msgstr "発行元 %s は正常に削除されました\n"

#: ../gui/modules/repository.py:844
#, python-format
msgid "Adding publisher %s\n"
msgstr "発行元 %s を追加しています\n"

#: ../gui/modules/repository.py:848
#, python-format
msgid "Updating publisher %s\n"
msgstr "発行元 %s を更新しています\n"

#: ../gui/modules/repository.py:852
#, python-format
msgid "Publisher %s succesfully added\n"
msgstr "発行元 %s は正常に追加されました\n"

#: ../gui/modules/repository.py:855
#, python-format
msgid "Publisher %s succesfully updated\n"
msgstr "発行元 %s は正常に更新されました\n"

#: ../gui/modules/repository.py:924
msgid "Disable Publisher:\n"
msgid_plural "Disable Publishers:\n"
msgstr[0] "発行元の無効化:\n"
msgstr[1] "発行元の無効化:\n"

#: ../gui/modules/repository.py:926
msgid "Enable Publisher:\n"
msgid_plural "Enable Publishers:\n"
msgstr[0] "発行元の有効化:\n"
msgstr[1] "発行元の有効化:\n"

#: ../gui/modules/repository.py:928
msgid "Remove Publisher:\n"
msgid_plural "Remove Publishers:\n"
msgstr[0] "発行元の削除:\n"
msgstr[1] "発行元の削除:\n"

#: ../gui/modules/repository.py:930
msgid "Change Priorities:\n"
msgstr "プロパティーの変更:\n"

#: ../gui/modules/repository.py:933
msgid "Apply the following change:"
msgid_plural "Apply the following changes:"
msgstr[0] "次の変更を適用します:"
msgstr[1] "次の変更を適用します:"

#: ../gui/modules/repository.py:983 ../gui/modules/repository.py:1040
msgid "Disabling"
msgstr "無効にする"

#: ../gui/modules/repository.py:985 ../gui/modules/repository.py:1042
msgid "Enabling"
msgstr "有効にする"

#: ../gui/modules/repository.py:988 ../gui/modules/repository.py:1045
#, python-format
msgid "%(enable)s publisher %(name)s\n"
msgstr "%(enable)s 発行元 %(name)s\n"

#: ../gui/modules/repository.py:1015
#, python-format
msgid "Changing priority for publisher %s\n"
msgstr "発行元 %s のプロパティーを変更しています\n"

#: ../gui/modules/repository.py:1210
msgid ""
"Mirrors for a secure publisher\n"
"need to begin with https://"
msgstr "セキュリティー保護されている発行元のミラーは\n"
       "https:// で始まる必要があります。"

#: ../gui/modules/repository.py:1218
msgid ""
"Origins for a secure publisher\n"
"need to begin with https://"
msgstr "セキュリティー保護されている発行元の起点は\n"
       "https:// で始まる必要があります。"

#: ../gui/modules/repository.py:1240
msgid "URI already added"
msgstr "URI はすでに追加されています"

#: ../gui/modules/repository.py:1487
msgid "Origin:\n"
msgid_plural "Origins:\n"
msgstr[0] "起点:\n"
msgstr[1] "起点:\n"

#: ../gui/modules/repository.py:1499
msgid "Description:\n"
msgstr "説明:\n"

#. More Generic for WebInstall
#: ../gui/modules/repository.py:1509
msgid "Publisher error"
msgstr "発行元エラー"

#: ../gui/modules/repository.py:1522
msgid "Specify SSL Key File"
msgstr "SSL キーのファイルを指定してください"

#: ../gui/modules/repository.py:1544
msgid "Specify SSL Certificate File"
msgstr "SSL 証明書のファイルを指定してください"

#: ../gui/modules/repository.py:1645
msgid "URI is not valid"
msgstr "URI が無効です"

#: ../gui/modules/repository.py:1659
msgid "SSL should not be specified"
msgstr "SSL は指定できません"

#: ../gui/modules/repository.py:1673
msgid "SSL Certificate not found at specified location"
msgstr "指定された場所には SSL 認証が見つかりません"

#: ../gui/modules/repository.py:1681
msgid "Mirror URI"
msgstr "ミラー URI"

#: ../gui/modules/repository.py:1690
msgid "Origin URI"
msgstr "起点 URI"

#: ../gui/modules/repository.py:1784
msgid "Enabling Publisher"
msgid_plural "Enabling Publishers"
msgstr[0] "発行元の有効化"
msgstr[1] "発行元の有効化"

#: ../gui/modules/repository.py:1786
msgid "Disabling Publisher"
msgid_plural "Disabling Publishers"
msgstr[0] "発行元の無効化"
msgstr[1] "発行元の無効化"

#: ../gui/modules/uarenamebe.py:58
msgid "The <b>libbe</b> library was not found on your system."
msgstr "<b>libbe</b> ライブラリがシステム上に見つかりませんでした。"

#: ../gui/modules/uarenamebe.py:65 ../gui/modules/uarenamebe.py:109
msgid "Rename BE"
msgstr "BE の名前変更"

#: ../gui/modules/uarenamebe.py:110
msgid "Renaming BE, please wait..."
msgstr "BE の名前を変更しています。しばらくお待ちください..."

#: ../gui/modules/uarenamebe.py:143 ../packagemanager.py:693
#, python-format
msgid "GUI will not respond to any event! %s. Check declare_signals()"
msgstr "GUI はどのイベントにも応答しません。%s。updatemanager.py のシグナルをチェックしてください"

#: ../gui/modules/uarenamebe.py:220
#, python-format
msgid ""
"Could not change the BE name to:\n"
"\t%s\n"
"\n"
"The following name will be used instead:\n"
"\t%s"
msgstr "次の BE 名に変更できませんでした:\n"
       "\t%s\n"
       "\n"
       "代わりに次の名前を使用します:\n"
       "\t%s"

#: ../gui/modules/uarenamebe.py:223
msgid "BE Name"
msgstr "BE 名"

#: ../gui/modules/uarenamebe.py:229
msgid ""
"Could not restart the system.\n"
"Please restart the system manually."
msgstr "システムを再起動できませんでした。\n"
       "システムを手動で再起動してください。"

#: ../gui/modules/uarenamebe.py:231
msgid "Restart Error"
msgstr "再起動エラー"

#: ../gui/modules/uarenamebe.py:245
msgid "This name already exists."
msgstr "この名前はすでに存在します。"

#: ../gui/modules/uarenamebe.py:251
msgid "BE name contains invalid character."
msgstr "BE 名に無効な文字が入っています。"

#: ../gui/modules/webinstall.py:131
#, python-format
msgid "GUI will not respond to any event! %s. Check webinstall.py signals"
msgstr "GUI はどのイベントにも応答しません。%s。webinstall.py のシグナルをチェックしてください"

#: ../gui/modules/webinstall.py:143
msgid ""
"\n"
" Add New Publisher\n"
msgid_plural ""
"\n"
" Add New Publishers\n"
msgstr[0] "\n  新規発行元を追加\n"
msgstr[1] "\n  新規発行元を追加\n"

#: ../gui/modules/webinstall.py:152
msgid "All specified publishers are already on the system."
msgstr "指定されたすべての発行元はすでにシステムに存在します。"

#: ../gui/modules/webinstall.py:162
#, python-format
msgid "\t%s "
msgstr "\t%s"

#: ../gui/modules/webinstall.py:166
#, python-format
msgid " (%s)\n"
msgstr " (%s)\n"

#: ../gui/modules/webinstall.py:172
msgid ""
"\n"
" Install Package\n"
msgid_plural ""
"\n"
" Install Packages\n"
msgstr[0] ""
       "\n"
       " パッケージをインストール\n"
msgstr[1] ""
       "\n"
       " パッケージをインストール\n"

#: ../gui/modules/webinstall.py:181
#, python-format
msgid "\t%s (disabled)\n"
msgstr "\t%s (disabled)\n"

#: ../gui/modules/webinstall.py:231
msgid "Click Proceed <b>only</b> if you trust this new publisher "
msgid_plural "Click Proceed <b>only</b> if you trust these new publishers "
msgstr[0] "この新しい発行元を信頼する場合<b>のみ</b>「続行」をクリックしてください"
msgstr[1] "これら新しい発行元を信頼する場合<b>のみ</b>「続行」をクリックしてください"

#: ../gui/modules/webinstall.py:302 ../gui/modules/webinstall.py:347
#: ../gui/modules/webinstall.py:391
msgid "Publisher Error"
msgstr "発行元エラー"

#: ../gui/modules/webinstall.py:310
msgid "The following publisher is disabled:\n"
msgid_plural "The following publishers are disabled:\n"
msgstr[0] "次の発行元は無効です:\n"
msgstr[1] "次の発行元は無効です:\n"

#: ../gui/modules/webinstall.py:314
#, python-format
msgid "\t<b>%s</b>\n"
msgstr "\t<b>%s</b>\n"

#: ../gui/modules/webinstall.py:317
msgid ""
"\n"
"Clicking OK will enable the publisher before proceeding with install. On "
"completion it will be disabled again."
msgid_plural ""
"\n"
"Clicking OK will enable the publishers before proceeding with install.\n"
"On completion they will be disabled again."
msgstr[0] "\n"
       "「了解」をクリックすると、インストールを続行する前に発行元を有効にします。インストールが完了すると、発行元は再び無効となります。"
msgstr[1] "\n"
       "「了解」をクリックすると、インストールを続行する前に発行元を有効にします。インストールが完了すると、発行元は再び無効となります。"

#: ../gui/modules/webinstall.py:329
msgid "Disabled Publisher"
msgid_plural "Disabled Publishers"
msgstr[0] "無効となっている発行元"
msgstr[1] "無効となっている発行元"

#: ../gui/modules/webinstall.py:387
#, python-format
msgid "Failed to add %s.\n"
msgstr "%s の追加に失敗しました。\n"

#: ../gui/modules/webinstall.py:388 ../packagemanager.py:1312
#: ../packagemanager.py:1324
msgid "No URI specified"
msgstr "URI が指定されていません"

#: ../gui/modules/webinstall.py:469
msgid "Web Installer Error"
msgstr "Web インストーラエラー"

#: ../modules/actions/__init__.py:95
#, python-format
msgid ""
"unknown action type '%(type)s' in package '%(fmri)s' in action '%(action)s'"
msgstr ""
       "アクション '%(action)s' のパッケージ '%(fmri)s のアクションタイプ '%(type)s' は不明です"

#: ../modules/actions/__init__.py:99
#, python-format
msgid "unknown action type '%(type)s' in action '%(action)s'"
msgstr "アクション '%(action)s' のアクションタイプ '%(type)s' は不明です"

#: ../modules/actions/__init__.py:112
#, python-format
msgid ""
"Malformed action in package '%(fmri)s' at position: %(pos)d:\n"
"    %(action)s\n"
"%(marker)s"
msgstr ""
       "パッケージ'%(fmri)s' の次の位置にあるアクションの形式が不正です: %(pos)d:\n"
       "    %(action)s\n"
       "%(marker)s"

#: ../modules/actions/__init__.py:117
#, python-format
msgid ""
"Malformed action at position: %(pos)d:\n"
"    %(action)s\n"
"%(marker)s"
msgstr ""
       "次の位置にあるアクションの形式が不正です: %(pos)d:\n"
       "    %(action)s\n"
       "%(marker)s"

#: ../modules/actions/__init__.py:144
#, python-format
msgid "invalid action in package %(fmri)s: %(action)s: %(error)s"
msgstr "パッケージ %(fmri)s のアクションが無効です: %(action)s: %(error)s"

#: ../modules/actions/__init__.py:147
#, python-format
msgid "invalid action, '%(action)s': %(error)s"
msgstr "無効なアクションです。'%(action)s': %(error)s"

#: ../modules/actions/__init__.py:178
#, python-format
msgid "required attribute '%s' was not provided."
msgstr "必須の属性 '%s' が指定されませんでした。"

#: ../modules/actions/__init__.py:253
#, python-format
msgid "required attribute, '%s', was not provided."
msgstr "必須の属性 '%s' が指定されませんでした。"

#: ../modules/actions/depend.py:65
msgid "Missing type attribute"
msgstr "タイプ属性がありません"

#: ../modules/actions/depend.py:69
msgid "Missing fmri attribute"
msgstr "fmri 属性がありません"

#: ../modules/actions/depend.py:73
#, python-format
msgid "Unknown type (%s) in depend action"
msgstr "依存アクションのタイプ (%s) は不明です"

#: ../modules/actions/directory.py:54 ../modules/actions/file.py:63
#: ../modules/actions/link.py:56
msgid "Empty path attribute"
msgstr "パス属性が空です"

#: ../modules/actions/generic.py:139
#, python-format
msgid "No such file: '%s'."
msgstr "ファイルがありません: '%s'。"

#: ../modules/actions/generic.py:142
#, python-format
msgid "'%s' is not a file."
msgstr "'%s' はファイルではありません。"

#: ../modules/actions/license.py:112
#, python-format
msgid "License file %s does not exist."
msgstr "ライセンスファイル %s が存在しません。"

#: ../modules/actions/license.py:117
#, python-format
msgid "Hash: '%(found)s' should be '%(expected)s'"
msgstr "ハッシュ: '%(found)s' は '%(expected)s' である必要があります"

#: ../modules/catalog.py:1514
#, python-format
msgid "Unknown info_needed type: %s"
msgstr "info_needed タイプが不明です: %s"

#. API consumer passed an unknown type for img_path.
#: ../modules/client/api.py:123
msgid "Unknown img_path type."
msgstr "img_path タイプが不明です。"

#: ../modules/client/api_errors.py:94
#, python-format
msgid ""
"Invalid package state change attempted '%(states)s' for package '%(fmri)s'."
msgstr "パッケージ '%(fmri)s' に対する 状態 '%(states)s' の変更は無効です。"

#: ../modules/client/api_errors.py:124
#, python-format
msgid ""
"Could not operate on %s\n"
"because of insufficient permissions. Please try the command again using "
"pfexec\n"
"or otherwise increase your privileges."
msgstr ""
       "アクセス権が十分でないため %s を\n"
       "操作できませんでした。"
       "pfexec\n"
       "を使用してコマンドを再試行するか、アクセス権を取得してください。"

#: ../modules/client/api_errors.py:129
msgid ""
"\n"
"Could not complete the operation because of insufficient permissions. "
"Please\n"
"try the command again using pfexec or otherwise increase your privileges.\n"
msgstr ""
       "\n"
       "アクセス権が十分でないため操作を完了できませんでした。"
       "pfexec を使用してコマンドを再試行するか、\n"
       "アクセス権を取得してください。\n"

#: ../modules/client/api_errors.py:140
#, python-format
msgid ""
"Could not operate on %s\n"
"because the file is in use. Please stop using the file and try the\n"
"operation again."
msgstr ""
       "ファイルが使用中のため %s を\n"
       "操作できませんでした。ファイルの使用を中止し、\n"
       "操作を再試行してください。"

#: ../modules/client/api_errors.py:155
#, python-format
msgid "Could not complete the operation on %s: read-only filesystem\n"
msgstr "%s 上での操作を完了できませんでした: ファイルシステムは読み取り専用です\n"

#: ../modules/client/api_errors.py:157
msgid "Could not complete the operation: read-only filesystem."
msgstr "操作を完了できませんでした:　ファイルシステムは読み取り専用です。"

#: ../modules/client/api_errors.py:184
msgid ""
"The following pattern(s) did not match any packages in the current catalog.\n"
"Try relaxing the pattern, refreshing and/or examining the catalogs:"
msgstr ""
       "次のパターンは、現在のカタログ内のどのパッケージにも一致しませんでした。\n"
       "パターンの条件緩和、再表示、カタログの検査のいずれか、あるいはそのすべてを試してください:"

#: ../modules/client/api_errors.py:191
msgid ""
"The following patterns only matched packages that are from publishers other "
"than that which supplied the already installed version of this package"
msgstr "次のパターンは、すでにインストール済みのこのパッケージのバージョンを提供した発行元とは異なる場所からのパッケージに一致します。"

#: ../modules/client/api_errors.py:205
#, python-format
msgid "'%s' matches no installed packages"
msgstr "'%s' はどのインストール済みパッケージにも一致しません"

#: ../modules/client/api_errors.py:209
#, python-format
msgid "'%s' is an illegal fmri"
msgstr "'%s' は不正な FMRI です"

#: ../modules/client/api_errors.py:213
#, python-format
msgid "'%s' supports the following architectures: %s"
msgstr "'%s' は次のアーキテクチャーをサポートしています: %s"

#: ../modules/client/api_errors.py:214
#, python-format
msgid "Image architecture is defined as: %s"
msgstr "イメージのアーキテクチャーは次のように定義されています: %s"

#: ../modules/client/api_errors.py:219
#, python-format
msgid "'%(p)s' depends on obsolete package '%(op)s'"
msgstr "'%(p)s' は廃止されたパッケージ '%(op)s' に依存しています"

#: ../modules/client/api_errors.py:223
msgid ""
"The proposed operation can not be performed for the following package(s) as "
"they are already installed: "
msgstr "次のパッケージがすでにインストールされているため、この操作は実行できません:"

#: ../modules/client/api_errors.py:230
msgid "The following different patterns specify thesame package(s):"
msgstr "次の異なるパターンは同じパッケージを指定します:"

#: ../modules/client/api_errors.py:240
msgid "No solution was found to satisfy constraints"
msgstr "制約を満たす解決は見つかりませんでした"

#: ../modules/client/api_errors.py:246
#, python-format
msgid "Package %s is missing a dependency: %s"
msgstr "パッケージ %s は依存性を欠いています: %s"

#: ../modules/client/api_errors.py:318
#, python-format
msgid ""
"The FMRI '%s' does not contain publisher information and cannot be used for "
"catalog operations."
msgstr "FMRI '%s' には発行元情報が含まれていないため、カタログ操作に使用できません。"

#: ../modules/client/api_errors.py:327
#, python-format
msgid ""
"Catalog meta_root '%(root)s' is invalid; unable to complete operation: '%(op)"
"s'."
msgstr "カタログの meta_root '%(root)s' は無効です; 操作を完了できません: '%(op)s'。"

#: ../modules/client/api_errors.py:344 ../modules/server/catalog.py:79
msgid "The following catalog files have incorrect permissions:\n"
msgstr "次のカタログファイルは不正なアクセス権を持っています:\n"

#: ../modules/client/api_errors.py:348 ../modules/server/catalog.py:83
#, python-format
msgid "\t%(fname)s: expected mode: %(emode)s, found mode: %(fmode)s\n"
msgstr "\t%(fname)s: 正しいモード: %(emode)s, 現在のモード: %(fmode)s\n"

#: ../modules/client/api_errors.py:358
#, python-format
msgid "The signature data for the '%s' catalog file is not valid."
msgstr "'%s' カタログファイルの署名日付は無効です。"

#: ../modules/client/api_errors.py:368
#, python-format
msgid ""
"Unable to determine the updates needed for  the current catalog using the "
"provided catalog update data in '%s'."
msgstr "'%s' で提供されているカタログの更新日付を使用して、現在のカタログに対する更新の必要性を決定できません。"

#: ../modules/client/api_errors.py:378
#, python-format
msgid ""
"Unable to perform '%(op)s' operation for catalog %(name)s; completion would "
"result in a duplicate entry for package '%(fmri)s'."
msgstr "カタログ %(name)s に対する '%(op)s' 操作を実行できません;　完了時にパッケージ '%(fmri)s' のエントリが重複する可能性があります。"

#: ../modules/client/api_errors.py:390
msgid "Catalog updates can only be applied to an on-disk catalog."
msgstr "カタログの更新は、ディスク上のカタログにのみ適用することができます。"

#: ../modules/client/api_errors.py:398
#, python-format
msgid "Catalog file '%s' is invalid."
msgstr "カタログファイル '%s' は無効です。"

#: ../modules/client/api_errors.py:406
#, python-format
msgid ""
"Unable to determine the updates needed for the catalog using the provided "
"catalog update data in '%s'. The specified catalog updates are for an older "
"version of the catalog and cannot be used."
msgstr "'%s' で提供されているカタログの更新日付を使用して、このカタログに対する更新の必要性を決定できません。指定されたカタログの更新は、古いバージョンのカタログに対するもので、使用できません。"

#: ../modules/client/api_errors.py:417
#, python-format
msgid "'%s' could not be found in the catalog."
msgstr "カタログ内に '%s' が見つかりません。"

#: ../modules/client/api_errors.py:425
#, python-format
msgid "Unknown catalog update type '%s'"
msgstr "不明なカタログ更新タイプ '%s' です"

#: ../modules/client/api_errors.py:433
#, python-format
msgid "Unrecognized, unknown, or invalid CatalogPart '%s'"
msgstr "認識できない、不明、または無効な CatalogPart '%s' です。"

#: ../modules/client/api_errors.py:464
msgid ""
"No matching package could be found for the following FMRIs in any of the "
"catalogs for the current publishers:\n"
msgstr ""
       "次の FMRI に一致するパッケージが、現在の発行元のどのカタログにも見つかり"
       "ませんでした:\n"

#: ../modules/client/api_errors.py:469
#, python-format
msgid "%s (pattern did not match)\n"
msgstr "%s (パターンが一致しませんでした)\n"

#: ../modules/client/api_errors.py:471
#, python-format
msgid "%s (publisher did not match)\n"
msgstr "%s (発行元が一致しませんでした)\n"

#: ../modules/client/api_errors.py:474
#, python-format
msgid "%s (version did not match)\n"
msgstr "%s (バージョンが一致しませんでした)\n"

#: ../modules/client/api_errors.py:515
#, python-format
msgid "The search at url %s returned no results."
msgstr "URL %s の検索には結果が返されませんでした"

#: ../modules/client/api_errors.py:529
msgid "Some servers failed to respond appropriately:\n"
msgstr "一部のサーバーが適切に応答できませんでした:\n"

#: ../modules/client/api_errors.py:531 ../modules/client/api_errors.py:540
#, python-format
msgid ""
"%(o)s:\n"
"%(msg)s\n"
msgstr "%(o)s:\n"
       "%(msg)s\n"

#: ../modules/client/api_errors.py:534
#, python-format
msgid "%s did not return a valid response.\n"
msgstr "%s は有効な応答を返しませんでした。\n"

#: ../modules/client/api_errors.py:537
msgid "Some servers don't support requested search operation:\n"
msgstr "一部のサーバーは、要求された検索操作をサポートしません:\n"

#: ../modules/client/api_errors.py:551
msgid ""
"Search performance is degraded.\n"
"Run 'pkg rebuild-index' to improve search speed."
msgstr "検索パフォーマンスが低下しています。\n"
       "'pkg rebuild-index' を実行して検索スピードを改善してください。"

#: ../modules/client/api_errors.py:565
msgid "Search server does not support the requested protocol:"
msgstr "検索サーバーは、要求されたプロトコルをサポートしてません:"

#: ../modules/client/api_errors.py:713
#, python-format
msgid ""
"The specified file is in an unrecognized format or does not contain valid "
"publisher information: %s"
msgstr ""
       "指定されたファイルは、認識できない形式であるか、有効な発行元情報を"
       "含んでいません: %s"

#: ../modules/client/api_errors.py:716
msgid ""
"The specified file is in an unrecognized format or does not contain valid "
"publisher information."
msgstr ""
       "指定されたファイルは、認識できない形式であるか、有効な発行元情報を"
       "含んでいません。"

#: ../modules/client/api_errors.py:725
msgid "Unsupported pkg(5) publisher information data format."
msgstr "pkg(5) 発行元情報としてサポートされていないデータ形式です。"

#: ../modules/client/api_errors.py:751
#, python-format
msgid ""
"Error encountered while retrieving data from '%s':\n"
"%s"
msgstr ""
       "'%s' からデータ取得中にエラーが発生しました:\n"
       "%s"

#: ../modules/client/api_errors.py:753
#, python-format
msgid "Error encountered while retrieving data from: %s"
msgstr "データの取得中にエラーが発生しました: %s"

#: ../modules/client/api_errors.py:765
#, python-format
msgid "'%s' is not a valid location."
msgstr "'%s' は有効な場所ではありません。"

#: ../modules/client/api_errors.py:777
#, python-format
msgid "'%s' is not a valid boot environment name."
msgstr "'%s' は有効なブート環境名ではありません。"

#: ../modules/client/api_errors.py:789
#, python-format
msgid "The boot environment '%s' already exists."
msgstr "ブート環境名 %s はすでに存在しています。"

#: ../modules/client/api_errors.py:798
msgid ""
"Boot environment naming during package install is not supported on this\n"
"version of OpenSolaris. Please image-update without the --be-name option."
msgstr ""
       "このバージョンの OpenSolaris では、パッケージのインストール中にブート環境の\n"
       "名前を指定することはできません。--be-name オプションを指定せずに image-update を実行してください。"

#: ../modules/client/api_errors.py:804
msgid "Unable to clone the current boot environment."
msgstr "現在のブート環境を複製できません。"

#: ../modules/client/api_errors.py:817
#, python-format
msgid ""
"A problem occurred while attempting to rename the boot environment\n"
"currently named %(orig)s to %(dest)s."
msgstr ""
       "ブート環境の名前を現在の %(orig)s から %(dest)s に\n"
       "変更しようとしたときに問題が発生しました。"

#: ../modules/client/api_errors.py:828
#, python-format
msgid "Unable to mount %(name)s at %(mt)s"
msgstr "%(name)s を %(mt)s にマウントできません"

#: ../modules/client/api_errors.py:837
msgid ""
"Naming a boot environment when operating on a non-live image is\n"
"not allowed."
msgstr ""
       "ライブ以外のイメージの操作中にブート環境の名前を指定することは\n"
       "許可されていません"

#: ../modules/client/api_errors.py:848
msgid "Info does not recognize the following options:"
msgstr "Info では次のオプションは認識されません:"

#: ../modules/client/api_errors.py:850
msgid " '"
msgstr " '"

#: ../modules/client/api_errors.py:850
msgid "'"
msgstr "'"

#: ../modules/client/api_errors.py:879
#, python-format
msgid ""
"Publisher meta_root '%(root)s' is invalid; unable to complete operation: '%"
"(op)s'."
msgstr "発行元の meta_root '%(root)s' は無効です; 操作を完了できません: '%(op)s'。"

#: ../modules/client/api_errors.py:888
#, python-format
msgid "'%s' is not a valid publisher name."
msgstr "'%s' は有効な発行元名ではありません。"

#: ../modules/client/api_errors.py:896
#, python-format
msgid ""
"'%(value)s' is not a valid value for repository attribute '%(attribute)s'."
msgstr ""
       "'%(value)s' はリポジトリ属性 '%(attribute)s' に有効な値ではありません。"

#: ../modules/client/api_errors.py:909
#, python-format
msgid "'%s' is not a valid repository collection type."
msgstr "'%s' は有効なリポジトリ収集タイプではありません。"

#: ../modules/client/api_errors.py:917
#, python-format
msgid "'%s' is not a valid URI."
msgstr "'%s' は有効な URI ではありません。"

#: ../modules/client/api_errors.py:925
#, python-format
msgid "'%s' is not a valid URI priority; integer value expected."
msgstr "'%s' は有効な URI 優先順位ではありません。整数値が必要です。"

#: ../modules/client/api_errors.py:937
#, python-format
msgid "'%s' is not a valid repository URI sort policy."
msgstr "'%s' は有効なリポジトリ URI ソートポリシーではありません。"

#: ../modules/client/api_errors.py:946
#, python-format
msgid "Publisher '%s' is disabled and cannot be used for packaging operations."
msgstr "発行元 '%s' は無効になっているため、パッケージ化の操作に使用することはできません。"

#: ../modules/client/api_errors.py:955
#, python-format
msgid "A publisher with the same name or alias as '%s' already exists."
msgstr "'%s' と同じ名前または同じ別名の発行元がすでに存在します。"

#: ../modules/client/api_errors.py:964
#, python-format
msgid ""
"A repository with the same name or origin URIs already exists for publisher "
"'%s'."
msgstr ""
       "発行元 '%s' には、同じ名前または同じ起点 URI のリポジトリが"
       "すでに存在します。"

#: ../modules/client/api_errors.py:973
#, python-format
msgid "Mirror '%s' already exists for the specified publisher."
msgstr "指定された発行元には、ミラー '%s' がすでに存在します。"

#: ../modules/client/api_errors.py:982
#, python-format
msgid "Origin '%s' already exists for the specified publisher."
msgstr "指定された発行元には、起点 '%s' がすでに存在します。"

#: ../modules/client/api_errors.py:991
#, python-format
msgid ""
"At least one origin is required for %s.  You must add a new origin before "
"attempting to remove the specified origin(s)."
msgstr "少なくとも 1 つの起点が %s に対して必要です。指定した起点を削除する前に新しい起点を追加する必要があります。"

#: ../modules/client/api_errors.py:1001
msgid "The preferred publisher cannot be removed."
msgstr "優先された発行元を削除することはできません。"

#: ../modules/client/api_errors.py:1008
msgid "Cannot search a repository before or after itself"
msgstr "リポジトリの前後でそのリポジトリ自体を検索することはできません"

#: ../modules/client/api_errors.py:1016
msgid "Cannot remove the selected repository for a publisher."
msgstr "発行元の選択されたリポジトリを削除できません。"

#: ../modules/client/api_errors.py:1025
#, python-format
msgid ""
"Publisher '%s' is disabled and cannot be set as the preferred publisher."
msgstr ""
       "発行元 '%s' は無効になっているため、優先された発行元として設定することはできません。"

#: ../modules/client/api_errors.py:1034
msgid ""
"The preferred publisher may not be disabled.  Another publisher must be set "
"as the preferred publisher before this publisher can be disabled."
msgstr "優先発行元は無効になっていない可能性があります。この発行元を無効にする前に、他の発行元を優先された発行元に設定する必要があります。"

#: ../modules/client/api_errors.py:1044
#, python-format
msgid "Unknown legal URI '%s'."
msgstr "正当な URI '%s' は不明です。"

#: ../modules/client/api_errors.py:1052
#, python-format
msgid "Unknown publisher '%s'."
msgstr "発行元 '%s' は不明です。"

#: ../modules/client/api_errors.py:1060
#, python-format
msgid "Unknown related URI '%s'."
msgstr "関連 URI '%s' は不明です。"

#: ../modules/client/api_errors.py:1068
#, python-format
msgid "Unknown repository '%s'."
msgstr "リポジトリ '%s' は不明です。"

#: ../modules/client/api_errors.py:1076
#, python-format
msgid "Unknown repository mirror '%s'."
msgstr "リポジトリのミラー '%s' は不明です。"

#: ../modules/client/api_errors.py:1090
#, python-format
msgid "Publisher '%s' has no repositories that support the '%s' operation."
msgstr "発行元 '%s' には、 '%s' 操作をサポートするリポジトリがありません。"

#: ../modules/client/api_errors.py:1098
#, python-format
msgid "Unknown repository origin '%s'"
msgstr "リポジトリの起点 '%s' は不明です。"

#: ../modules/client/api_errors.py:1109
#, python-format
msgid "The URI '%(uri)s' contains an unsupported scheme '%(scheme)s'."
msgstr "URI '%(uri)s' には、サポートされていないスキーマ '%(scheme)s' が含まれています。"

#: ../modules/client/api_errors.py:1112
msgid "The specified URI contains an unsupported scheme."
msgstr "指定された URI には、サポートされていないスキーマが含まれています。"

#: ../modules/client/api_errors.py:1120
#, python-format
msgid "'%(attr)s' is not supported for '%(scheme)s'."
msgstr "'%(attr)s' は '%(scheme)s' でサポートされていません。"

#: ../modules/client/api_errors.py:1147
#, python-format
msgid ""
"Certificate '%(cert)s' for publisher '%(pub)s' needed to access '%(uri)s', "
"has expired.  Please install a valid certificate."
msgstr ""
       "'%(uri)s' へのアクセスに必要な発行元 '%(pub)s' の証明書 '%(cert)s' の"
       "有効期限が切れています。有効な証明書をインストールしてください。"

#: ../modules/client/api_errors.py:1152
#, python-format
msgid ""
"Certificate '%(cert)s' for publisher '%(pub)s', has expired.  Please install "
"a valid certificate."
msgstr ""
       "発行元 '%(pub)s' の証明書 '%(cert)s' の有効期限が切れています。"
       "有効な証明書をインストールしてください。"

#: ../modules/client/api_errors.py:1157
#, python-format
msgid ""
"Certificate '%(cert)s', needed to access '%(uri)s', has expired.  Please "
"install a valid certificate."
msgstr ""
       "'%(uri)s' へのアクセスに必要な証明書 '%(cert)s' の有効期限が切れています。"
       "有効な証明書をインストールしてください。"

#: ../modules/client/api_errors.py:1160
#, python-format
msgid "Certificate '%s' has expired.  Please install a valid certificate."
msgstr "証明書 '%s' の有効期限が切れています。有効な証明書をインストールしてください。"

#: ../modules/client/api_errors.py:1173
#, python-format
msgid ""
"Certificate '%(cert)s' for publisher '%(pub)s', needed to access '%(uri)s', "
"will expire in '%(days)s' days."
msgstr ""
       "'%(uri)s' へのアクセスに必要な発行元 '%(pub)s' の証明書 '%(cert)s' は、"
       "あと '%(days)s' 日で期限切れになります。"

#: ../modules/client/api_errors.py:1178
#, python-format
msgid ""
"Certificate '%(cert)s' for publisher '%(pub)s' will expire in '%(days)s' "
"days."
msgstr ""
       "発行元 '%(pub)s' の証明書 '%(cert)s' は、あと '%(days)s' 日で期限切れに"
       "なります。"

#: ../modules/client/api_errors.py:1182
#, python-format
msgid ""
"Certificate '%(cert)s', needed to access '%(uri)s', will expire in '%(days)"
"s' days."
msgstr ""
       "'%(uri)s' へのアクセスに必要な証明書 '%(cert)s' は、あと '%(days)s' 日で期限切れに"
       "なります。"

#: ../modules/client/api_errors.py:1185
#, python-format
msgid "Certificate '%(cert)s' will expire in '%(days)s' days."
msgstr "証明書 '%(cert)s' は、あと '%(days)s' 日で期限切れになります。"

#: ../modules/client/api_errors.py:1197
#, python-format
msgid ""
"Certificate '%(cert)s' for publisher '%(pub)s', needed to access '%(uri)s', "
"is invalid."
msgstr ""
       "'%(uri)s' へのアクセスに必要な発行元 '%(pub)s' の証明書 '%(cert)s' が"
       "無効です。"

#: ../modules/client/api_errors.py:1201
#, python-format
msgid "Certificate '%(cert)s' for publisher '%(pub)s' is invalid."
msgstr "発行元 '%(pub)s' の証明書 '%(cert)s' が無効です。"

#: ../modules/client/api_errors.py:1205
#, python-format
msgid "Certificate '%(cert)s' needed to access '%(uri)s' is invalid."
msgstr "'%(uri)s' へのアクセスに必要な証明書 '%(cert)s' が無効です。"

#: ../modules/client/api_errors.py:1208
#, python-format
msgid "Invalid certificate '%s'."
msgstr "証明書 '%s' が無効です。"

#: ../modules/client/api_errors.py:1219
#, python-format
msgid ""
"Unable to locate key '%(key)s' for publisher '%(pub)s' needed to access '%"
"(uri)s'."
msgstr "'%(uri)s' へのアクセスに必要な発行元 '%(pub)s' のキー '%(key)s' が見つかりません。"

#: ../modules/client/api_errors.py:1223
#, python-format
msgid "Unable to locate key '%(key)s' for publisher '%(pub)s'."
msgstr "発行元 '%(pub)s' のキー '%(key)s' が見つかりません。"

#: ../modules/client/api_errors.py:1227
#, python-format
msgid "Unable to locate key '%(key)s' needed to access '%(uri)s'."
msgstr "'%(uri)s' へのアクセスに必要なキー '%(key)s' が見つかりません。"

#: ../modules/client/api_errors.py:1230
#, python-format
msgid "Unable to locate key '%s'."
msgstr "キー key '%s' が見つかりません。"

#: ../modules/client/api_errors.py:1241
#, python-format
msgid ""
"Unable to locate certificate '%(cert)s' for publisher '%(pub)s' needed to "
"access '%(uri)s'."
msgstr ""
       "'%(uri)s' へのアクセスに必要な発行元 '%(pub)s' の証明書 '%(cert)s' が"
       "見つかりません。"

#: ../modules/client/api_errors.py:1246
#, python-format
msgid "Unable to locate certificate '%(cert)s' for publisher '%(pub)s'."
msgstr "発行元 '%(pub)s' の証明書 '%(cert)s' が見つかりません。"

#: ../modules/client/api_errors.py:1250
#, python-format
msgid "Unable to locate certificate '%(cert)s' needed to access '%(uri)s'."
msgstr "'%(uri)s' へのアクセスに必要な証明書 '%(cert)s' が見つかりません。"

#: ../modules/client/api_errors.py:1253
#, python-format
msgid "Unable to locate certificate '%s'."
msgstr "証明書 '%s' が見つかりません。"

#: ../modules/client/api_errors.py:1265
#, python-format
msgid ""
"Certificate '%(cert)s' for publisher '%(pub)s', needed to access '%(uri)s', "
"has a future effective date."
msgstr ""
       "'%(uri)s' へのアクセスに必要な発行元 '%(pub)s' の証明書 '%(cert)s' の発効日が"
       "未来の日付になっています。"

#: ../modules/client/api_errors.py:1270
#, python-format
msgid ""
"Certificate '%(cert)s' for publisher '%(pub)s' has a future effective date."
msgstr ""
       "発行元 '%(pub)s' の証明書 '%(cert)s' の発効日が未来の日付になっています。"

#: ../modules/client/api_errors.py:1274
#, python-format
msgid ""
"Certificate '%(cert)s' needed to access '%(uri)s' has a future effective "
"date."
msgstr ""
       "'%(uri)s' へのアクセスに必要な証明書 '%(cert)s' の発効日が"
       "未来の日付になっています。"

#: ../modules/client/api_errors.py:1277
#, python-format
msgid "Certificate '%s' has a future effective date."
msgstr "証明書 '%s' の発効日が未来の日付になっています。"

#: ../modules/client/api_errors.py:1290
#, python-format
msgid "Gave a bad response:%s"
msgstr "不正な応答でした:%s"

#: ../modules/client/api_errors.py:1301
#, python-format
msgid "Could not find %s"
msgstr "%s が見つかりませんでした"

#: ../modules/client/api_errors.py:1324
#, python-format
msgid "The signature data for the manifest of the '%s' package is not valid."
msgstr "'%s' パッケージのマニフェストに対する署名日付が無効です。"

#: ../modules/client/api_errors.py:1326
msgid "The signature data for the manifest is not valid."
msgstr "マニフェストに対する署名日付が無効です。"

#: ../modules/client/api_errors.py:1339
#, python-format
msgid ""
"there is already an image at: %s.\n"
"To override, use the -f (force) option."
msgstr "次の場所にすでにイメージが存在します: %s。\n"
       "上書きするには、-f (force) オプションを使用します。"

#: ../modules/client/api_errors.py:1345
#, python-format
msgid ""
"the specified image path is not empty: %s.\n"
"To override, use the -f (force) option."
msgstr "指定されたイメージのパスは空ではありません: %s。\n"
       "上書きするには、-f (force) オプションを使用します。"

#: ../modules/client/bootenv.py:134
msgid "pkg: unable to create an auto snapshot. pkg recovery is disabled."
msgstr "pkg: 自動スナップショットを作成できません。pkg の復元が無効になっています。"

#: ../modules/client/bootenv.py:267
#, python-format
msgid "pkg: A system error %(e)s was caught executing %(cmd)s"
msgstr "pkg: システムエラー %(e)s が %(cmd)s の実行中に検出されました"

#: ../modules/client/bootenv.py:272
#, python-format
msgid ""
"pkg: '%(cmd)s' failed. \n"
"with a return code of %(ret)d."
msgstr "pkg: '%(cmd)s' が失敗しました。\n リターンコードは %(ret)d です。"

#: ../modules/client/bootenv.py:284
#, python-format
msgid "pkg: unable to activate %s"
msgstr "pkg: %s をアクティブにすることができません"

#: ../modules/client/bootenv.py:294
#, python-format
msgid "pkg: unable to unmount %s"
msgstr "pkg: %s をマウント解除できません"

#: ../modules/client/bootenv.py:300
#, python-format
msgid ""
"\n"
"A clone of %s exists and has been updated and activated.\n"
"On the next boot the Boot Environment %s will be mounted on '/'.\n"
"Reboot when ready to switch to this updated BE.\n"
msgstr "\n %s のクローンが存在しており、それが更新およびアクティブ化されました。\n 次回リブート時にはブート環境 %s が '/' にマウントされます。\n 準備が整ったらリブートしてこの更新済みの BE に切り替えてください。\n"

#: ../modules/client/bootenv.py:314
#, python-format
msgid "%s has been updated successfully"
msgstr "%s の更新に成功しました"

#: ../modules/client/bootenv.py:346
#, python-format
msgid ""
" The running system has not been modified. Modifications were only made to a "
"clone of the running system.  This clone is mounted at %s should you wish to "
"inspect it."
msgstr ""
       " 稼働中のシステムは変更されませんでした。変更が加えられたのは、"
       "稼働中のシステムのクローンに対してだけです。このクローンは %s にマウント"
       "されているので、必要に応じて調査を行なってください。"

#: ../modules/client/bootenv.py:357
#, python-format
msgid "pkg: unable to rollback BE %s and restore image"
msgstr "pkg: BE %s をロールバックしてイメージを復元できません"

#: ../modules/client/bootenv.py:367
#, python-format
msgid "%s failed to be updated. No changes have been made to %s."
msgstr "%s の更新に失敗しました。%s は変更されていません。"

#: ../modules/client/bootenv.py:380
#, python-format
msgid "pkg: unable to destroy snapshot %s"
msgstr "pkg: スナップショット %s を破棄できません"

#: ../modules/client/bootenv.py:409
#, python-format
msgid "pkg: unable to create BE %s"
msgstr "pkg: BE %s を作成できません"

#: ../modules/client/bootenv.py:414
#, python-format
msgid "pkg: unable to mount BE %(name)s on %(clone_dir)s"
msgstr "pkg: BE %(name)s を %(clone_dir)s にマウントできません"

#: ../modules/client/bootenv.py:420
#, python-format
msgid ""
"The Boot Environment %(name)s failed to be updated. A snapshot was taken "
"before the failed attempt and is mounted here %(clone_dir)s. Use 'beadm "
"unmount %(clone_name)s' and then 'beadm activate %(clone_name)s' if you wish "
"to boot to this BE."
msgstr "ブート環境 %(name)s の更新に失敗しました。失敗した試行を行う前に取られたスナップショットが"
       "存在しており、それが %(clone_dir)s にマウントされています。この BE にブートする場合は、"
       "'beadm unmount %(clone_name)s' を使用してから 'beadm activate %(clone_name)s' を使用してください。"

#: ../modules/client/bootenv.py:435
#, python-format
msgid ""
"The Boot Environment %s failed to be updated. A snapshot was taken before "
"the failed attempt and has been restored so no changes have been made to %s."
msgstr ""
       "ブート環境 %s の更新に失敗しました。失敗した試行を行う前に取られた"
       "スナップショットが復元されたため、%s は変更されていません。"

#: ../modules/client/image.py:2418
msgid "SUNWipkg update check failed."
msgstr "SUNWipkg の更新チェックが失敗しました。"

#: ../modules/client/imageplan.py:169
#, python-format
msgid "Planning for %s failed: %s\n"
msgstr "%s に対する計画が失敗しました: %s\n"

#: ../modules/client/imageplan.py:846
msgid "Removal Phase"
msgstr "削除フェーズ"

#: ../modules/client/imageplan.py:856
msgid "Install Phase"
msgstr "インストールフェーズ"

#: ../modules/client/imageplan.py:867
msgid "Update Phase"
msgstr "更新フェーズ"

#: ../modules/client/imageplan.py:923
msgid "Index Phase"
msgstr "インデックスフェーズ"

#: ../modules/client/pkg_solver.py:162
msgid "This version excluded by specified installation version"
msgstr "このバージョンは指定されたインストールのバージョンから除外されます"

#: ../modules/client/pkg_solver.py:171
#, python-format
msgid "This version is excluded by installed incorporation %s"
msgstr "このバージョンはインストールされた結合 %s から除外されます"

#: ../modules/client/pkg_solver.py:181
#, python-format
msgid "This version is excluded by freeze %s"
msgstr "このバージョンは凍結 %s から除外されます"

#: ../modules/client/pkg_solver.py:197
#, python-format
msgid "No matching version of %s can be installed:"
msgstr "%s の一致していないバージョンはインストールできません:"

#: ../modules/client/pkg_solver.py:229
#, python-format
msgid "No version of %s can be installed:"
msgstr "%s のバージョンはインストールできません:"

#: ../modules/client/pkg_solver.py:917
msgid "Publisher differs from installed or specifed version"
msgstr "発行元はインストールまたは指定されたバージョンとは異なります"

#: ../modules/client/pkg_solver.py:927
msgid "Publisher is lower ranked"
msgstr "発行元はより低くランク付けされています"

#: ../modules/client/pkg_solver.py:962
#, python-format
msgid "Newer version %s is already installed"
msgstr "より新しいバージョン %s がすでにインストール済みです"

#: ../modules/client/pkg_solver.py:971
msgid "Package doesn't support image architecture"
msgstr "パッケージはイメージ構造をサポートしていません"

#: ../modules/client/pkg_solver.py:973
#, python-format
msgid "Package doesn't support image variant %s"
msgstr "パッケージはイメージ形式 %s をサポートしていません"

#: ../modules/client/progress.py:589
#, python-format
msgid "Retrieving catalog '%s'..."
msgstr "カタログ '%s' を取得しています..."

#: ../modules/client/progress.py:596
msgid "Caching catalogs ..."
msgstr "カタログをキャッシュしていています..."

#: ../modules/client/progress.py:602
msgid "Loading catalog cache ..."
msgstr "カタログキャッシュを読み込んでいます..."

#: ../modules/client/progress.py:608
msgid "Refreshing catalog"
msgstr "カタログを再表示しています"

#: ../modules/client/publisher.py:957
#, python-format
msgid ""
"\n"
"Unable to retrieve package data for publisher '%(prefix)s' from one\n"
"of the following origin(s):\n"
"\n"
"%(origins)s\n"
"\n"
"The catalog retrieved from one of the origin(s) listed above only\n"
"contains package data for: %(pubs)s.\n"
msgstr "\n"
       "発行元 '%(prefix)s' のパッケージデータを\n"
       "次の起点のいずれからも検出できません:\n"
       "\n"
       "%(origins)s\n"
       "\n"
       "上記の起点一覧の 1 つから検出されたカタログには\n"
       "次の日付のパッケージデータのみが含まれます: %(pubs)s.\n"

#: ../modules/client/publisher.py:969
#, python-format
msgid ""
"This is either a result of invalid origin information being provided\n"
"for publisher '%s', or because the wrong publisher name was\n"
"provided when this publisher was added.\n"
msgstr "これは発行元 '%s' に関する誤った起点情報によるものか\n"
       "この発行元が追加されたときに誤った発行元名をつけたことに\n"
       "よるものです。\n"

#: ../modules/client/publisher.py:977
#, python-format
msgid ""
"To resolve this issue, correct the origin information provided for\n"
"publisher '%(prefix)s' using the pkg set-publisher subcommand, or re-add\n"
"the publisher using the correct name and remove the '%(prefix)s'\n"
"publisher.\n"
msgstr "この問題を解決するには、pkg set-publisher サブコマンドを使用して\n"
       "発行元 '%(prefix)s' の起点情報を修正するか、正しい名前で発行元を再度追加してから\n"
       "発行元 '%(prefix)s' を削除します\n"

#: ../modules/client/publisher.py:985
#, python-format
msgid ""
"To re-add this publisher with the correct name, execute the following\n"
"commands as a privileged user:\n"
"\n"
"pkg set-publisher -P -g %(origin)s %(pub)s\n"
"pkg unset-publisher %(prefix)s\n"
msgstr "この発行元を正しい名前で再度追加するには、特権ユーザーとして次のコマンドを実行します:\n"
       "\n"
       "pkg set-publisher -P -g %(origin)s %(pub)s\n"
       "pkg unset-publisher %(prefix)s\n"

#: ../modules/client/publisher.py:994
msgid ""
"The origin(s) listed above contain package data for more than one\n"
"publisher, but this issue can likely be resolved by executing one\n"
"of the following commands as a privileged user:\n"
msgstr "上記の起点一覧は複数の発行元のパッケージデータを含んでいます。\n"
       "ただしこの問題は特権ユーザーとして次のコマンドのいずれかを実行することで\n"
       "解決できるでしょう:\n"

#: ../modules/client/publisher.py:1001
#, python-format
msgid "pkg set-publisher -P -g %(origin)s %(pub)s\n"
msgstr "pkg set-publisher -P -g %(origin)s %(pub)s\n"

#: ../modules/client/publisher.py:1005
#, python-format
msgid ""
"Afterwards, the old publisher should be removed by executing the\n"
"following command as a privileged user:\n"
"\n"
"pkg unset-publisher %s\n"
msgstr "その後、特権ユーザーとして次のコマンドを実行することで\n"
       "旧発行元を削除できます:\n"
       "\n"
       "pkg unset-publisher %s\n"

#: ../modules/file_layout/file_manager.py:73
#, python-format
msgid ""
"The FileManager cannot %(cre)s %(ent)s because it is configured read-only."
msgstr "読み取り専用に構成されているため、FileManager は %(ent)s を %(cre)s できません。"

#: ../modules/file_layout/file_manager.py:93
#, python-format
msgid "FileManager was unable to create %s or the directories containing it."
msgstr "FileManager は、%s またはそれを含むディレクトリを作成できませんでした。"

#: ../modules/file_layout/file_manager.py:112
#, python-format
msgid ""
"The following paths were found but cannot be accounted for by any of the "
"known layouts:\n"
"%s"
msgstr "次のパスが見つかりましたが、どの既知のレイアウトによっても説明できません:\n"
       "%s"

#: ../modules/flavor/base.py:51
#, python-format
msgid "Couldn't find %s"
msgstr "%s が見つかりません"

#: ../modules/flavor/base.py:106
#, python-format
msgid "Subclasses of Dependency must implement dep_key. Current class is %s"
msgstr "依存関係のサブクラスには、dep_key を実装する必要があります。"
       "現在のクラスは %s"

#: ../modules/flavor/elf.py:46
#, python-format
msgid "%s had this elf error:%s"
msgstr "%s には次の elf エラーがあります:%s"

#: ../modules/flavor/elf.py:60
#, python-format
msgid ""
"%(pp)s (which will be installed at %(ip)s) had this token, %(tok)s, in its "
"run path: %(rp)s.  It is not currently possible to automatically expand this "
"token. Please specify its value on the command line."
msgstr "%(pp)s (%(ip)s にインストールされる) は、実行パス: %(rp)s にトークン %(tok)s があります。現時点でこのトークンを自動的に拡張できません。コマンド行上で値を指定してください。"

#: ../modules/flavor/python.py:43
#, python-format
msgid "Could not find the file for %(name)s imported in %(localpath)s"
msgstr "%(localpath)s にインポートされた %(name)s のファイルが見つかりません"

#: ../modules/fmri.py:97
#, python-format
msgid "FMRI '%s' is missing version information."
msgstr "FMRI '%s' はバージョン情報がありません。"

#: ../modules/indexer.py:158
msgid "Reading Existing Index"
msgstr "既存インデックスの読み取り"

#: ../modules/indexer.py:673 ../modules/indexer.py:698
msgid "Indexing Packages"
msgstr "パッケージのインデックス作成"

#: ../modules/manifest.py:462
#, python-format
msgid ""
"%(fp)s:\n"
"%(e)s"
msgstr "%(fp)s:\n"
       "%(e)s"

#: ../modules/manifest.py:473
#, python-format
msgid ""
"%(fp)s contains an action which is missing the expected attribute: %(at)s.\n"
"The action is:%(act)s"
msgstr "%(fp)s は、必要な属性 %(at)s が不足しているアクションを含んでいます。\n"
       "アクションは %(act)s です"

#: ../modules/manifest.py:566
#, python-format
msgid "Attribute value '%s' not 'true' or 'false'"
msgstr "属性の値 '%s' は 'true' または 'false' ではありません"

#: ../modules/misc.py:455
msgid "B"
msgstr "B"

#: ../modules/misc.py:456
msgid "kB"
msgstr "KB"

#: ../modules/misc.py:457
msgid "MB"
msgstr "MB"

#: ../modules/misc.py:458
msgid "GB"
msgstr "GB"

#: ../modules/misc.py:459
msgid "TB"
msgstr "TB"

#: ../modules/misc.py:460
msgid "PB"
msgstr "PB"

#: ../modules/misc.py:461 ../modules/misc.py:465
msgid "EB"
msgstr "EB"

#: ../modules/p5i.py:87
msgid "missing version"
msgstr "不足しているバージョン"

#: ../modules/p5i.py:89
msgid "invalid version"
msgstr "無効なバージョン"

#: ../modules/publish/dependencies.py:60
#, python-format
msgid ""
"The file dependency %s has paths which resolve to multiple packages. The "
"actions are as follows:\n"
"%s"
msgstr "ファイル依存性 %s には複数のパッケージを解決するパスがあります。アクションは次の通りです:\n"
       "%s"

#: ../modules/publish/dependencies.py:73
#, python-format
msgid ""
"The file dependency %s depends on a path delivered by multiple packages. "
"Those packages are:%s"
msgstr "ファイル依存性 %s は複数のパッケージによって渡されるパスに依存しています。"
       "これらのパッケージは次のとおりです: %s"

#: ../modules/publish/dependencies.py:87
#, python-format
msgid ""
"%s has unresolved dependency '%s' under the following combinations of "
"variants:\n"
"%s"
msgstr "%s には次のバリエーションの組み合わせを行なった場合、未解決の依存性 '%s' があります:\n"
       "%s"

#: ../modules/publish/transaction.py:78
#, python-format
msgid "Unsupported scheme '%(scheme)s' in URL: '%(url)s'."
msgstr "URL '%(url)s' のスキーマ '%(scheme)s' はサポートされていません。"

#: ../modules/publish/transaction.py:82
#, python-format
msgid "Malformed URL: '%s'."
msgstr "不正な形式の URL です: '%s'。"

#: ../modules/publish/transaction.py:83
#, python-format
msgid "Invalid repository URL: '%(url)s': %(msg)s"
msgstr "不正なリポジトリ URL です: '%(url)s': %(msg)s"

#: ../modules/publish/transaction.py:95
#, python-format
msgid ""
"'%(op)s' failed for transaction ID '%(trans_id)s'; status '%(status)s': %"
"(msg)s"
msgstr "'%(op)s' はトランザクション ID '%(trans_id)s' に失敗しました; 状態 '%(status)s': %(msg)s"

#: ../modules/publish/transaction.py:102
#, python-format
msgid "'%(op)s' failed for transaction ID '%(trans_id)s': %(msg)s"
msgstr "'%(op)s' はトランザクション ID '%(trans_id)s' に失敗しました: %(msg)s"

#: ../modules/publish/transaction.py:108
#, python-format
msgid ""
"'%(op)s' failed; unable to initiate transaction:\n"
"%(msg)s"
msgstr "'%(op)s' は失敗しました; トランザクションを初期化できません:\n "
       "%(msg)s"

#: ../modules/publish/transaction.py:111
#, python-format
msgid ""
"Unable to initiate transaction:\n"
"%s"
msgstr "トランザクションを初期化できません:\n"
       "%s"

#: ../modules/publish/transaction.py:125
#, python-format
msgid ""
"Unsupported operation '%(op)s' for the specified repository type '%(type)s'."
msgstr "指定されたリポジトリタイプ '%(type)s' に対してサポートされていない操作 '%(op)s' です。"

#: ../modules/publish/transaction.py:147
msgid "Not an absolute path."
msgstr "絶対パスではありません。"

#: ../modules/publish/transaction.py:156
#, python-format
msgid ""
"An error occurred while trying to initialize the repository directory "
"structures:\n"
"%s"
msgstr "リポジトリのディレクトリ構成を初期化しようとしているときにエラーが発生しました:\n"
       "%s"

#: ../modules/publish/transaction.py:294
msgid "Unsupported or temporarily unavailable operation requested."
msgstr "サポートされていないか、一時的に使用できない操作が要求されました。"

#: ../modules/publish/transaction.py:422
#, python-format
msgid "Unknown failure; no transaction ID provided in response: %s"
msgstr "不明なエラー; 応答にトランザクション ID が指定されていません: %s"

#: ../modules/query_parser.py:126
#, python-format
msgid "An unparseable character in query at position : %d"
msgstr "クエリーの次のポジションに解析できない文字があります: %d"

#: ../modules/query_parser.py:337
#, python-format
msgid "In query %(query)s, %(name)s had a bad value of '%(bv)s'."
msgstr "クエリー %(query)s の %(name)s に誤った値 '%(bv)s' があります。"

#: ../modules/query_parser.py:352
#, python-format
msgid ""
"A query is expected to have five fields: case sensitivity, return type, "
"number of results to return, the number at which to start returning results, "
"and the text of the query.  The query provided lacked at least one of those "
"fields:\n"
"%s"
msgstr "クエリーには、大文字小文字を区別するかどうか、戻り値のタイプ、戻り値の数、どの結果から返し始めるかを示す番号、クエリーのテキストの 5 つのフィールドが必要です。このクエリーには、これらのフィールドが 1 つ以上不足しています:\n"
       "%s"

#: ../modules/query_parser.py:374
msgid "Could not parse query."
msgstr "クエリーを構文解析できません。"

#: ../modules/query_parser.py:375
#, python-format
msgid "Problem occurred with: %s"
msgstr "次の箇所で問題が発生しました: %s"

#: ../modules/query_parser.py:514
msgid "This expression produces action results:"
msgstr "この式で生成される結果はアクションです:"

#: ../modules/query_parser.py:516
msgid "This expression produces package results:"
msgstr "この式で生成される結果はパッケージです:"

#: ../modules/query_parser.py:519
msgid ""
"'AND' and 'OR' require those expressions to produce the same type of results."
msgstr "'AND' と 'OR' では、これらの式から 同じタイプの結果が生成される必要があります。"

#: ../modules/search_errors.py:135
#, python-format
msgid ""
"Got an empty unquote_list while indexing. split_chars was %(sc)s and line "
"was %(l)s"
msgstr "インデックス作成中に空の unquote_list を取得しました。split_chars は %(sc)s、行は %(l)s でした"

#: ../modules/search_errors.py:147
#, python-format
msgid ""
"Had an empty line in the main dictionary. split_chars is %(sc)s and "
"unquote_list is %(ul)s.%(s)s"
msgstr "主辞書内に空の行があります。split_chars は %(sc)s、unquote_list は %(ul)s です。%(s)s"

#: ../modules/server/catalog.py:225 ../modules/server/repository.py:795
#, python-format
msgid ""
"Index corrupted or out of date. Removing old index directory (%s)  and "
"rebuilding search indexes."
msgstr ""
       "インデックスが壊れているか古くなっています。古いインデックスディレクトリ (%s) を削除して"
       "検索インデックスを再構築しています。"

#: ../modules/server/catalog.py:241 ../modules/server/repository.py:811
msgid "writable root not writable by current user id or group."
msgstr "現在のユーザー ID またはグループでは書き込みできない書き込み可能ルートです。"

#: ../modules/server/catalog.py:246 ../modules/server/repository.py:814
msgid "unable to write to index directory."
msgstr "インデックスディレクトリに書き込めません。"

#: ../modules/server/depot.py:395
msgid "Directory listing not allowed."
msgstr "ディレクトリの表示はできません。"

#: ../modules/server/depot.py:678
#, python-format
msgid "The specified Action Type, '%s', is not valid."
msgstr "指定されたアクションタイプ、'%s' は無効です。"

#: ../modules/server/depot.py:941
msgid "No matching FMRI found in repository catalog."
msgstr "リポジトリのカタログに、一致しない FMRI  が見つかりました。"

#: ../modules/server/repository.py:78
#, python-format
msgid "The specified catalog file '%s', could not be found."
msgstr "指定されたカタログファイル '%s' が見つかりません。"

#: ../modules/server/repository.py:87
#, python-format
msgid "No file could be found for the specified hash name: '%s'."
msgstr "指定されたハッシュ名 '%s' に対するファイルは見つかりませんでした。"

#: ../modules/server/repository.py:96
#, python-format
msgid "The specified repository root '%s' does not contain a valid repository."
msgstr "指定されたリポジトリルート '%s' は有効なリポジトリを含んでいません。"

#: ../modules/server/repository.py:108
#, python-format
msgid "The specified Transaction ID '%s' is invalid."
msgstr "指定されたトランザクション ID '%s' は無効です。"

#: ../modules/server/repository.py:116
#, python-format
msgid "No manifest could be found for the FMRI: '%s'."
msgstr "FMRI '%s' に対するマニフェストが見つかりません。"

#: ../modules/server/repository.py:125
msgid ""
"The requested operation cannot be performed when the repository is used in "
"mirror mode."
msgstr "リポジトリをミラーモードで使用する場合、要求された操作は実行できません。"

#: ../modules/server/repository.py:134
msgid "The repository is read-only and cannot be modified."
msgstr "リポジトリは読み取り専用で、修正はできません。"

#: ../modules/server/repository.py:143
msgid "No token was provided to search."
msgstr "検索にトークンが指定されていませんでした。"

#: ../modules/server/repository.py:145
#, python-format
msgid "The specified search token '%s' is invalid."
msgstr "指定された検索トークン '%s' は不正です。"

#: ../modules/server/repository.py:153
msgid "Search functionality is temporarily unavailable."
msgstr "検索機能は一時的に使用できません。"

#: ../modules/server/repository.py:162
msgid ""
"The format of the repository or its contents needs to be upgraded before it "
"can be used to serve package data.  However, it is currently read-only and "
"cannot be upgraded.  If using pkg.depotd, please restart the server without "
"read-only so that the repository can be upgraded."
msgstr "リポジトリのフォーマット、またはコンテンツは、パッケージデータを供給する前にアップグレードする必要があります。しかし、現在読み取り専用となっているため、アップグレードできません。pkg.depotd を使用している場合は、読み取り専用をはずしてサーバーを再起動するとリポジトリがアップグレードされます。"

#. To upgrade the repository, the catalog will have to
#. be rebuilt.
#: ../modules/server/repository.py:432
msgid "Upgrading repository; this process will take some time."
msgstr "リポジトリをアップグレードしています; この処理には時間がかかります。"

#: ../modules/server/repository.py:541
msgid "repository directories incomplete"
msgstr "リポジトリのディレクトリは不完全です"

#: ../modules/server/repository.py:550
msgid ""
"repository directories not writeable by current user id or group and are "
"incomplete"
msgstr "リポジトリのディレクトリは、現在のユーザー ID またはグループで書き込みできないため、不完全です"

#. Don't add packages with corrupt
#. manifests to the catalog.
#: ../modules/server/repository.py:663
#, python-format
msgid "Skipping %(fmri)s; invalid manifest: %(error)s"
msgstr "%(fmri)s をスキップしています; マニフェストが無効です: %(error)s"

#: ../modules/server/repositoryconfig.py:406
#, python-format
msgid ""
"Unable to locate or read the specified repository configuration file: '%s'."
msgstr "指定されたリポジトリの構成ファイル '%s' を読み込み、または読み取りできません。"

#: ../modules/server/transaction.py:72
#, python-format
msgid "Unrecognized or malformed data in operation payload: '%s'."
msgstr "オペレーションのペイロードに認識できないデータ、または不正な形式のデータ '%s' があります。"

#: ../modules/server/transaction.py:89
#, python-format
msgid "The specified client_release is invalid: '%s'"
msgstr "指定された client_release は無効です: '%s'"

#: ../modules/server/transaction.py:92
#, python-format
msgid "'The specified FMRI, '%s', has an invalid version."
msgstr "指定された FMRI '%s' は無効なバージョンです。"

#: ../modules/server/transaction.py:95
#, python-format
msgid "The specified FMRI, '%s', already exists or has been restricted."
msgstr "指定された FMRI '%s' はすでに存在するか、制限されています。"

#: ../modules/server/transaction.py:98
#, python-format
msgid ""
"The specified FMRI, '%s', must include the publisher prefix as the "
"repository contains package data for more than one publisher or a default "
"publisher has not been defined."
msgstr "そのリポジトリには複数の発行元のパッケージデータがあるか、デフォルトの発行元を定義していないため、指定された FMRI '%s' には発行元の接頭辞を含む必要があります。"

#: ../modules/server/transaction.py:104
#, python-format
msgid "The specified FMRI, '%s', is invalid."
msgstr "指定された FMRI '%s' は無効です。"

#: ../modules/server/transaction.py:113
#, python-format
msgid "Transaction ID '%s' is already open."
msgstr "トランザクション ID '%s' はすでに開いています。"

#: ../modules/server/transaction.py:428
msgid "An obsolete package cannot contain actions other than 'set'."
msgstr "廃止パッケージには 'set' 以外のアクションを含むことができません。"

#: ../modules/server/transaction.py:436
msgid "A renamed package cannot contain actions other than 'set' and 'depend'."
msgstr "名前を変更したパッケージには、'set' と 'depend' 以外のアクションを含むことができません。"

#: ../modules/server/transaction.py:451
msgid "A package may not  be marked for both obsoletion and renaming."
msgstr "パッケージは、廃止と名前変更の両方をマークすることはできない場合があります。"

#: ../modules/server/transaction.py:454
#, python-format
msgid "A '%s' action cannot be present in an obsolete package: %s"
msgstr "'%s' アクションは廃止パッケージには使用できません: %s"

#: ../modules/server/transaction.py:458
#, python-format
msgid "A '%s' action cannot be present in a renamed package: %s"
msgstr "'%s' アクションは名前を変更したパッケージには使用できません: %s"

#: ../modules/server/transaction.py:478
msgid "A renamed package must contain at least one 'depend' action."
msgstr "名前を変更したパッケージには、'depend' アクションを 1 つ以上含む必要があります。"

#: ../packagemanager.py:322
msgid "All Packages"
msgstr "すべてのパッケージ"

#: ../packagemanager.py:324
msgid "Installed Packages"
msgstr "インストール済みパッケージ"

#: ../packagemanager.py:328
msgid "Not Installed Packages"
msgstr "インストールされていないパッケージ"

#: ../packagemanager.py:332
msgid "Selected Packages"
msgstr "選択したパッケージ"

#: ../packagemanager.py:335
msgid "All Publishers (Installed)"
msgstr "すべての発行元 (インストール済み)"

#: ../packagemanager.py:336
msgid "All Publishers (Search)"
msgstr "すべての発行元 (検索)"

#: ../packagemanager.py:337
msgid "Add..."
msgstr "追加..."

#: ../packagemanager.py:785
msgid "Export Selections Confirmation"
msgstr "エクスポート選択項目の確認"

#: ../packagemanager.py:787
msgid "<b>Export the following to a Web Install .p5i file:</b>"
msgstr "<b>次の項目を Web Install .p5i ファイルにエクスポートします:</b>"

#: ../packagemanager.py:829
msgid "Export Selections Error"
msgstr "エクスポート選択項目エラー"

#: ../packagemanager.py:840
msgid "Export Selections"
msgstr "エクスポート選択項目"

#: ../packagemanager.py:847
msgid "p5i Files"
msgstr "p5i ファイル"

#: ../packagemanager.py:851
msgid "All Files"
msgstr "すべてのファイル"

#: ../packagemanager.py:856
msgid "my_packages"
msgstr "my_packages"

#: ../packagemanager.py:903 ../packagemanager.py:911
msgid "Search (Ctrl-F)"
msgstr "検索 (Ctrl-F)"

#: ../packagemanager.py:1058
#, python-format
msgid ""
"<html><head></head><body><H2>Welcome toPackageManager!</H2><br><font "
"color='#0000FF'>Warning: Unable to load Start Page:<br>%s</font></body></"
"html>"
msgstr ""
       "<html><head></head><body><H2>パッケージマネージャーへようこそ</H2><br><font "
       "color='#0000FF'>警告: 開始ページを読み込めません:<br>%s</font></body></"
       "html>"

#: ../packagemanager.py:1070
msgid "failed to respond"
msgstr "応答に失敗しました"

#: ../packagemanager.py:1073
msgid "invalid response"
msgstr "無効な応答です"

#: ../packagemanager.py:1074
msgid "A valid response was not returned."
msgstr "有効な応答が返されました。"

#: ../packagemanager.py:1077
msgid "unsupported search"
msgstr "サポートされていない検索です"

#: ../packagemanager.py:1151
#, python-format
msgid "Loading... %s"
msgstr "読み込み中... %s"

#: ../packagemanager.py:1157
msgid "Stopped"
msgstr "停止しました"

#: ../packagemanager.py:1171
msgid "Done"
msgstr "完了"

#: ../packagemanager.py:1178
#, python-format
msgid ""
"<html><head></head><body><font color='#000000'>                    <a "
"href='stub'></a></font>                    <a href='pm?%s=internal&uri=%"
"s'>                    <IMG SRC = '%s/startpage_star."
"png'                     style='border-style: none'></a> "
"<br><br>                    <h2><font color='#0000FF'>Warning: Unable "
"to                     load URL</font></h2><br>%s</body></html>"
msgstr "<html><head></head><body><font color='#000000'> <a "
       "href='stub'></a></font> <a href='pm?%s=internal&uri=%"
       "s'> <IMG SRC = '%s/startpage_star."
       "png' style='border-style: none'></a>"
       "<br><br> <h2><font color='#0000FF'>警告: URL を読み込めません</font></h2><br>%s</body></html>"

#: ../packagemanager.py:1244
#, python-format
msgid "View packages in %(s1)s%(pub)s%(e1)s"
msgstr "%(s1)s%(pub)s%(e1)s のパッケージを表示"

#: ../packagemanager.py:1252
#, python-format
msgid "Search within %(s1)sAll Publishers%(e1)s"
msgstr "%(s1)sすべての発行元%(e1)sを検索"

#: ../packagemanager.py:1259
#, python-format
msgid "View installed packages for %(s1)sAll Publishers%(e1)s"
msgstr "%(s1)s すべての発行元 %(e1)s を表示"

#: ../packagemanager.py:1266
#, python-format
msgid "Change View to %(s1)sAll Packages%(e1)s"
msgstr "表示を%(s1)sすべての発行元%(e1)sに変更"

#: ../packagemanager.py:1274
#, python-format
msgid "Display %(s1)sSearch Help%(e1)s"
msgstr "%(s1)s検索ヘルプ%(e1)sを表示"

#: ../packagemanager.py:1277
#, python-format
msgid "Loading %(s1)sSearch Help%(e1)s ..."
msgstr "%(s1)s検索ヘルプ%(e1)sを読み込み中..."

#: ../packagemanager.py:1287
#, python-format
msgid "View results for %s"
msgstr "%s の結果を表示します"

#: ../packagemanager.py:1346
msgid "Empty Action not supported"
msgstr "空のアクションはサポートされていません"

#: ../packagemanager.py:1349
#, python-format
msgid "Action not supported: %s"
msgstr "アクションはサポートされていません: %s"

#: ../packagemanager.py:1448
msgid "Click to toggle selections"
msgstr "選択を切り替えるには、クリックしてください"

#: ../packagemanager.py:1463 ../packagemanager.py:3105
msgid "Status"
msgstr "状態"

#: ../packagemanager.py:1605
msgid "all selection toggle"
msgstr "すべての選択を切り替え"

#: ../packagemanager.py:1657
msgid "Not Installed"
msgstr "インストールされていません"

#: ../packagemanager.py:1659 ../packagemanager.py:3123
msgid "Updates Available"
msgstr "更新が使用可能です"

#. Only enable the first section. Later other sections are enabled
#. in __add_category_to_section() if the section contains any categories
#. which in turn contain some packages.
#: ../packagemanager.py:1780 ../packagemanager.py:4803
msgid "All Categories"
msgstr "すべてのカテゴリ"

#: ../packagemanager.py:1782
msgid "Meta Packages"
msgstr "メタパッケージ"

#: ../packagemanager.py:1783
msgid "Applications"
msgstr "アプリケーション"

#: ../packagemanager.py:1784
msgid "Desktop (GNOME)"
msgstr "デスクトップ (GNOME)"

#: ../packagemanager.py:1785
msgid "Development"
msgstr "開発"

#: ../packagemanager.py:1786
msgid "Distributions"
msgstr "ディストリビューション"

#: ../packagemanager.py:1787
msgid "Drivers"
msgstr "ドライバ"

#: ../packagemanager.py:1789
msgid "Web Services"
msgstr "Web サービス"

#: ../packagemanager.py:1912
msgid "Search all publishers"
msgstr "すべての発行元を検索"

#: ../packagemanager.py:1914
msgid "Search current publisher"
msgstr "現在の発行元を検索"

#: ../packagemanager.py:1962
msgid ""
"alt='[Information]' title='Information' ALIGN='bottom'></"
"TD><TD><h3><b>Search All Publishers</b></h3><TD></TD></TR><TR><TD></TD><TD> "
"Use the Search field to search for packages within the following Publishers:"
"</TD></TR>"
msgstr "alt='[情報]' title='情報' ALIGN='bottom'></"
       "TD><TD><h3><b>すべての発行元を検索</b></h3><TD></TD></TR><TR><TD></TD><TD> "
       "次の発行元のパッケージを検索するには、「検索」フィールドを使用してください:"
       "</TD></TR>"

#: ../packagemanager.py:1980
msgid ""
"<TR><TD></TD><TD></TD></TR><TR><TD></TD><TD>Click on the Publishers below to "
"view their list of packages:</TD></TR>"
msgstr "<TR><TD></TD><TD></TD></TR><TR><TD></TD><TD>次の発行元をクリックして "
       "その発行元のパッケージを表示してください:</TD></TR>"

#: ../packagemanager.py:1994
msgid ""
"alt='[Information]' title='Information' ALIGN='bottom'></"
"TD><TD><h3><b>Search in All Publishers (Installed)</b></h3><TD></TD></"
"TR><TR><TD></TD><TD> Search is <b>not</b> supported in All Publishers "
"(Installed).</TD></TR>"
msgstr "alt='[Information]' title='Information' ALIGN='bottom'></"
       "TD><TD><h3><b>すべての発行元 (インストール済み)を検索</b></h3><TD></TD></"
       "TR><TR><TD></TD><TD> 検索はすべての発行元 (インストール済み) には<b>サポートされていません</b> 。</TD></TR>"

#: ../packagemanager.py:2000 ../packagemanager.py:2069
#: ../packagemanager.py:2095
msgid ""
"<TR><TD></TD><TD<TD></TD></TR><TR><TD></TD><TD<TD></TD></TR><TR><TD></"
"TD><TD<TD><b>Suggestions:</b><br></TD></TR><TR><TD></TD><TD<TD>"
msgstr "<TR><TD></TD><TD<TD></TD></TR><TR><TD></TD><TD<TD></TD></TR><TR><TD></"
       "TD><TD<TD><b>推奨:</b><br></TD></TR><TR><TD></TD><TD<TD>"

#: ../packagemanager.py:2005
#, python-format
msgid ""
"<li style='padding-left:7px'>Return to view packages for All Publishers <a "
"href='pm?pm-action=internal&search=%s'>(Installed)</a></li>"
msgstr "<li style='padding-left:7px'>すべての発行元"
       "<a href='pm?pm-action=internal&search=%s'>(インストール済み)</a>"
       "のパッケージ表示に戻る</li>"

#: ../packagemanager.py:2008
#, python-format
msgid ""
"<li style='padding-left:7px'>Search for <b>%(text)s</b> using All Publishers "
"<a href='pm?pm-action=internal&search=%(all_pubs)s'>(Search)</a></li>"
msgstr "<li style='padding-left:7px'>すべての発行元を使用<a href='pm?pm-action=internal&search=%(all_pubs)s'>(検索)</a>して<b>%(text)s</b>を検索</li>"

#: ../packagemanager.py:2013 ../packagemanager.py:2136
#: ../packagemanager.py:2165
#, python-format
msgid ""
"<li style='padding-left:7px'>See <a href='pm?pm-action=internal&search=%"
"s'>Search Help</a></li></TD></TR>"
msgstr "<li style='padding-left:7px'><a href='pm?pm-action=internal&search=%"
       "s'>検索ヘルプ</a>を参照</li></TD></TR>"

#: ../packagemanager.py:2023
msgid ""
"alt='[Information]' title='Information' ALIGN='bottom'></"
"TD><TD><h3><b>Recent Searches</b></h3><TD></TD></TR><TR><TD></TD><TD> Access "
"stored results from recent searches in this session.</TD></TR>"
msgstr "alt='[Information]' title='Information' ALIGN='bottom'></"
       "TD><TD><h3><b>最近の検索結果</b></h3><TD></TD></TR><TR><TD></TD><TD>保存されているこのセッション中の最近の検索結果にアクセスする。</TD></TR>"

#: ../packagemanager.py:2036
msgid "results"
msgstr "結果"

#: ../packagemanager.py:2042
msgid "Click on the search results link below to view the stored results:"
msgid_plural ""
"Click on one of the search results links below to view the stored results:"
msgstr[0] "保存してある検索結果を表示するには、下の検索結果リンクをクリックします:"
msgstr[1] "保存してある検索結果の 1 つを表示するには、下の検索結果リンクをクリックします:"

#: ../packagemanager.py:2057
msgid ""
"alt='[Information]' title='Information' ALIGN='bottom'></TD><TD><h3><b>View "
"Packages</b></h3><TD></TD></TR><TR><TD></TD><TD>"
msgstr "alt='[Information]' title='Information' ALIGN='bottom'></TD><TD><h3><b>パッケージの表示</b></h3><TD></TD></TR><TR><TD></TD><TD>"

#: ../packagemanager.py:2060
#, python-format
msgid ""
"There is one package in this category, however it is not visible in the "
"selected View:\n"
"<li style='padding-left:7px'><b>%s</b></li>"
msgid_plural ""
"There are a number of packages in this category, however they are not "
"visible in the selected View:\n"
"<li style='padding-left:7px'><b>%s</b></li>"
msgstr[0] "このカテゴリ内にはパッケージが 1 つありますが、選択した表示には現れません:\n"
       "<li style='padding-left:7px'><b>%s</b></li>"
msgstr[1] "このカテゴリ内には数多くのパッケージが ありますが、選択した表示には現れません:\n"
       "<li style='padding-left:7px'><b>%s</b></li>"

#: ../packagemanager.py:2073 ../packagemanager.py:2099
#, python-format
msgid ""
"<li style='padding-left:7px'><a href='pm?pm-action=internal&search=%"
"s'>Change View to All Packages</a></li>"
msgstr "<li style='padding-left:7px'><a href='pm?pm-action=internal&search=%"
       "s'>表示を「すべてのパッケージ」に変更する</a></li>"

#: ../packagemanager.py:2084
msgid ""
"alt='[Information]' title='Information' ALIGN='bottom'></"
"TD><TD><h3><b>Search Results</b></h3><TD></TD></TR><TR><TD></TD><TD>"
msgstr "alt='[情報]' title='情報' ALIGN='bottom'></"
       "TD><TD><h3><b>検索結果</b></h3><TD></TD></TR><TR><TD></TD><TD>"

#: ../packagemanager.py:2087
#, python-format
msgid ""
"Found <b>%(num)s</b> package matching <b>%(text)s</b> in All Packages, "
"however it is not listed in the <b>%(filter)s</b> View."
msgid_plural ""
"Found <b>%(num)s</b> packages matching <b>%(text)s</b> in All Packages, "
"however they are not listed in the <b>%(filter)s</b> View."
msgstr[0] "すべてのパッケージでは <b>%(num)s</b> パッケージが <b>%(text)s</b> に一致しましたが、 "
       "<b>%(filter)s</b> 表示では表示されていません。"
msgstr[1] "すべてのパッケージでは <b>%(num)s</b> パッケージが <b>%(text)s</b> に一致しましたが、 "
       "<b>%(filter)s</b> 表示では表示されていません。"

#: ../packagemanager.py:2119
#, python-format
msgid ""
"alt='[Information]' title='Information' ALIGN='bottom'></"
"TD><TD><h3><b>Search Results</b></h3><TD></TD></TR><TR><TD></TD><TD>No "
"packages found in <b>%(pub)s</b> matching <b>%(text)s</b></TD></TR>"
msgstr "alt='[情報]' title='情報' ALIGN='bottom'></"
       "TD><TD><h3><b>検索結果</b></h3><TD></TD></TR><TR><TD></TD><TD><b>%(pub)s</b> には "
       "<b>%(text)s</b> に一致するパッケージが見つかりませんでした</TD></TR>"

#: ../packagemanager.py:2124
msgid ""
"<TR><TD></TD><TD<TD></TD></TR><TR><TD></TD><TD<TD></TD></TR><TR><TD></"
"TD><TD<TD><b>Suggestions:</b><br></TD></TR><TR><TD></TD><TD<TD><li "
"style='padding-left:7px'>Check your spelling</li><li style='padding-"
"left:7px'>Try new search terms</li>"
msgstr "<TR><TD></TD><TD<TD></TD></TR><TR><TD></TD><TD<TD></TD></TR><TR><TD></"
       "TD><TD<TD><b>推奨:</b><br></TD></TR><TR><TD></TD><TD<TD><li "
       "style='padding-left:7px'>スペルを確認</li><li style='padding-"
       "left:7px'>別の検索語句を試す</li>"

#: ../packagemanager.py:2131
#, python-format
msgid ""
"<li style='padding-left:7px'>Search for <b>%(text)s</b> within <a href='pm?"
"pm-action=internal&search=%(all_pubs)s'>All Publishers</a></li>"
msgstr "<li style='padding-left:7px'><a href='pm?pm-action=internal&search=%(all_pubs)s'>"
       "すべての発行元</a>で <b>%(text)s</b> を検索</li>"

#: ../packagemanager.py:2153
#, python-format
msgid ""
"<table border='0' cellpadding='3' style='table-layout:fixed' ><TR><TD><IMG "
"SRC = '%s/dialog-warning.png' style='border-style: none' alt='[Warning]' "
"title='Warning' ALIGN='bottom'></TD><TD><h3><b>Search Warning</b></h3><TD></"
"TD></TR><TR><TD></TD><TD>Search using only the wildcard character, <b>*</b>, "
"is not supported in All Publishers</TD></TR>"
msgstr "<table border='0' cellpadding='3' style='table-layout:fixed' ><TR><TD><IMG "
       "SRC = '%s/dialog-warning.png' style='border-style: none' alt='[Warning]' "
       "title='Warning' ALIGN='bottom'></TD><TD><h3><b>検索警告</b></h3><TD></"
       "TD></TR><TR><TD></TD><TD>ワイルドカード文字<b>*</b>のみを使用することは「すべての発行元」でサポートされていません</TD></TR>"

#: ../packagemanager.py:2160
msgid ""
"<TR><TD></TD><TD<TD></TD></TR><TR><TD></TD><TD<TD></TD></TR><TR><TD></"
"TD><TD<TD><b>Suggestions:</b><br></TD></TR><TR><TD></TD><TD<TD><li "
"style='padding-left:7px'>Try new search terms</li>"
msgstr "<TR><TD></TD><TD<TD></TD></TR><TR><TD></TD><TD<TD></TD></TR><TR><TD></TD><TD<TD><b>推奨:</b><br></TD></TR><TR><TD></TD><TD<TD><li  style='padding-left:7px'>別の検索語句を試す</li>"

#: ../packagemanager.py:2408 ../packagemanager.py:2429
#: ../packagemanager.py:4873
msgid "All Publishers"
msgstr "すべての発行元"

#: ../packagemanager.py:2620
msgid "Search cleared"
msgstr "検索は消去されました"

#: ../packagemanager.py:2640
msgid "Fetching legal information..."
msgstr "使用条件情報を取得しています..."

#: ../packagemanager.py:3347 ../packagemanager.py:3701
msgid "Refreshing package catalog information"
msgstr "パッケージのカタログ情報を更新しています"

#: ../packagemanager.py:3394
msgid "All"
msgstr "すべて"

#. Installed
#: ../packagemanager.py:3477
msgid "<b>Upgradeable Version:</b>"
msgstr "<b>アップグレード可能なバージョン:</b>"

#. Installed and up to date
#: ../packagemanager.py:3486
msgid "Installed package is up-to-date"
msgstr "インストールされているパッケージは最新の状態です。"

#: ../packagemanager.py:3569
#, python-format
msgid ""
"The latest version of %s cannot be installed. It may be possible to do so "
"after updating your system. Run \"Updates\" to get your system up-to-date."
msgstr "%s の最新バージョンはインストールできません。システムを更新したあとで可能になるかもしれません。システムを最新の状態にするには \"Updates\" を実行します。"

#: ../packagemanager.py:3574
#, python-format
msgid ""
"Cannot upgrade to the latest version of %s. It may be possible to do so "
"after updating your system. Run \"Updates\" to get your system up-to-date."
msgstr "%s の最新バージョンはインストールできません。システムを更新したあとで可能になるかもしれません。システムを最新の状態にするには \"Updates\" を実行します。"

#: ../packagemanager.py:3594
msgid "Latest Version: "
msgstr "最新バージョン:"

#: ../packagemanager.py:3965
msgid "Selected for Removal:"
msgstr "選択された削除対象:"

#: ../packagemanager.py:3973
msgid "Selected for Install/Update:"
msgstr "選択されたインストール/更新対象:"

#: ../packagemanager.py:3977
msgid "Select packages by marking the checkbox and click to Install/Update."
msgstr "チェックボックスにチェックマークを付けてパッケージを選択し、「インストール/更新」をクリックします。"

#: ../packagemanager.py:3981
msgid "Select packages by marking the checkbox and click to Remove selected."
msgstr "チェックボックスにチェックマークを付けてパッケージを選択し、「選択したパッケージを削除します」をクリックします。"

#: ../packagemanager.py:4043
msgid "Fetching information..."
msgstr "情報を取得しています..."

#: ../packagemanager.py:4115
msgid "(not installed)"
msgstr "(未インストール)"

#: ../packagemanager.py:4158
msgid "Name\tDependency\tInstalled Version\n"
msgstr "名前\t依存関係\tインストールされているバージョン\n"

#: ../packagemanager.py:4199
msgid ""
"\n"
"This might be caused by network problem while accessing the repository."
msgstr ""
       "\n"
       "リポジトリへのアクセス中にネットワークの問題が発生すると、このようになる場合があります。"

#: ../packagemanager.py:4202
msgid "Files Details not available for this package..."
msgstr "このパッケージのファイルの詳細がありません..."

#: ../packagemanager.py:4205
msgid "Dependencies info not available for this package..."
msgstr "このパッケージの依存性の情報がありません..."

#: ../packagemanager.py:4208
msgid "Information not available for this package..."
msgstr "このパッケージの情報がありません..."

#: ../packagemanager.py:4255
msgid "Not available"
msgstr "使用不可"

#: ../packagemanager.py:4263
msgid "License could not be shown due to conversion problem."
msgstr "変換に問題があるため、ライセンスは表示できませんでした。"

#: ../packagemanager.py:4607 ../packagemanager.py:4656
msgid "Loading package list"
msgstr "パッケージリストを読み込んでいます"

#. This can happen if the repository does not
#. contain any packages
#: ../packagemanager.py:4620
msgid "Selected publisher does not contain any packages."
msgstr "選択した発行元にパッケージが含まれていません。"

#: ../packagemanager.py:4851
msgid "Recent Searches"
msgstr "最近の検索履歴"

#: ../packagemanager.py:5273
#, python-format
msgid "Total: %(total)s  Selected: %(selected)s"
msgstr "合計: %(total)s  選択: %(selected)s"

#: ../packagemanager.py:5293
#, python-format
msgid ""
"Searching %(s1)s%(active_pub)s%(e1)s for %(s1)s%(search_text)s%(e1)s ..."
msgstr "%(s1)s%(active_pub)s%(e1)s で %(s1)s%(search_text)s%(e1)s を検索しています..."

#: ../packagemanager.py:5299
#, python-format
msgid "Searching for %(s1)s%(search_text)s%(e1)s ..."
msgstr "%(s1)s%(search_text)s%(e1)s を検索しています..."

#: ../packagemanager.py:5304
#, python-format
msgid "%(number)d packages found matching %(s1)s%(search_text)s%(e1)s"
msgstr "%(s1)s%(search_text)s%(e1)s に一致するパッケージが %(number)d 個見つかりました"

#: ../packagemanager.py:5423
msgid ""
"Network problem.\n"
"\n"
msgstr ""
       "ネットワークの問題。\n"
       "\n"

#: ../packagemanager.py:5424
msgid "Details:\n"
msgstr "詳細:\n"

#: ../packagemanager.py:5425
#, python-format
msgid "%s/%s catalogs successfully updated:\n"
msgstr "%s/%s 個のカタログが正常に更新されました:\n"

#: ../packagemanager.py:5505
msgid "Save selected..."
msgstr "選択した項目を保存..."

#: ../packagemanager.py:5506
msgid "Save selected packages..."
msgstr "選択したパッケージを保存..."

#: ../packagemanager.py:5507
msgid ""
"The installed package(s) require a reboot before installation can be "
"completed."
msgstr ""
       "パッケージのインストールを完了するには、リブートする"
       "必要があります。"

#: ../packagemanager.py:5544
msgid "Unable to initialize gtk"
msgstr "gtk を初期化できません"

#: ../pkgdep.py:75
msgid ""
"Usage:\n"
"        pkgdepend [options] command [cmd_options] [operands]\n"
"\n"
"Subcommands:\n"
"        pkgdepend generate [-DIkMm] manifest proto_dir\n"
"        pkgdepend [options] resolve [-dMos] manifest ...\n"
"\n"
"Options:\n"
"        -R dir\n"
"        --help or -?\n"
"Environment:\n"
"        PKG_IMAGE"
msgstr "使用方法:\n"
       "pkgdepend [options] command [cmd_options] [operands]\n"
       "\n"
       "サブコマンド:\n"
       "pkgdepend generate [-DIkMm] manifest proto_dir\n"
       "pkgdepend [options] resolve [-dMos] manifest ...\n"
       "\n"
       "オプション:\n"
       "-R dir\n"
       "--help or -?\n"
       "環境:\n"
       "PKG_IMAGE"

#: ../pkgdep.py:115
msgid "-D arguments must be of the form 'name=value'."
msgstr "引数 -D は 'name=value' の形式になっている必要があります。"

#: ../pkgdep.py:134
msgid "Generate only accepts exactly two arguments."
msgstr "「生成」は 2 つの引数のみ受け付けます。"

#: ../pkgdep.py:140
msgid ""
"ORIGIN may not be specified using -D. It will be inferred from the\n"
"install paths of the files."
msgstr "ORIGIN は -D を使用して指定しなくても構いません。\n"
       "それはファイルのインストールパスから推測されます。"

#: ../pkgdep.py:149 ../pkgdep.py:212
#, python-format
msgid "The manifest file %s could not be found."
msgstr "マニフェストファイル %s が見つかりません。"

#: ../pkgdep.py:153
#, python-format
msgid "The proto directory %s could not be found."
msgstr "プロトディレクトリ %s が見つかりません。"

#: ../pkgdep.py:160
#, python-format
msgid ""
"Could not parse manifest %(manifest)s because of the following line:\n"
"%(line)s"
msgstr "次の行が原因で、マニフェスト %(manifest)s の構文解析ができませんでした:\n"
       "%(line)s"

#: ../pkgdep.py:206
msgid "-o cannot be used with -d or -s"
msgstr "-o は -d または -s と同時に使用できません"

#: ../pkgdep.py:218
#, python-format
msgid "The output directory %s is not a directory."
msgstr "出力先ディレクトリ %s はディレクトリではありません。"

#: ../pkgdep.py:248
#, python-format
msgid "'%s' is not an install image"
msgstr "'%s' はインストールイメージではありません"

#: ../pkgdep.py:254
msgid "Could not parse one or more manifests because of the "
msgstr "次の原因で、1 つ以上のマニフェストの構文解析ができませんでした"

#: ../pkgdep.py:323 ../pkgdep.py:358
#, python-format
msgid "Could not open %s to echo manifest"
msgstr "マニフェストを表示するために %s を開けませんでした"

#: ../pkgdep.py:328
msgid ""
"\n"
"\n"
msgstr "\n"
       "\n"

#: ../pkgdep.py:350
#, python-format
msgid "Could not open output file %s for writing"
msgstr "出力ファイル %s を書き込みのために開くことができませんでした"

#: ../pkgdep.py:392
#, python-format
msgid "Out dir %s does not exist and could not be created. Error is: %s"
msgstr "出力先のディレクトリ %s が存在せず、作成もできませんでした。"
       "エラーは次のとおりです: %s"

#: ../pkgdep.py:467
msgid "generate subcommand doesn't use -R"
msgstr "generate サブコマンドは -R を使用しません"

#: ../pkgdep.py:485
#, python-format
msgid ""
"The %(cmd)s command appears out of sync with the libraries provided \n"
"by SUNWipkg. The client version is %(client)s while the library API version "
"is %(api)s"
msgstr ""
       "%(cmd)s コマンドは、SUNWipkg で提供されているライブラリと同期がとれていない\n"
       "ようです。クライアントのバージョンが %(client)s であるのに対して、ライブラリ API "
       "のバージョンは %(api)s です。"

#: ../publish.py:80
msgid ""
"Usage:\n"
"        pkgsend [options] command [cmd_options] [operands]\n"
"\n"
"Packager subcommands:\n"
"        pkgsend create-repository --set-property <section.property>=<value>\n"
"        pkgsend open [-en] pkg_fmri\n"
"        pkgsend add action arguments\n"
"        pkgsend import [-T file_pattern] bundlefile ...\n"
"        pkgsend include [-d basedir] .. [manifest] ...\n"
"        pkgsend close [-A | [--no-index] [--no-catalog]]\n"
"        pkgsend publish [ -d basedir] ... [--no-index] \n"
"          [--fmri-in-manifest | pkg_fmri] [--no-catalog] [manifest] ...\n"
"        pkgsend generate [-T file_pattern] bundlefile ....\n"
"        pkgsend refresh-index\n"
"\n"
"Options:\n"
"        -s repo_uri     target repository URI\n"
"        --help or -?    display usage message\n"
"\n"
"Environment:\n"
"        PKG_REPO"
msgstr "使用方法:\n"
       "pkgsend [options] command [cmd_options] [operands]\n"
       "\n"
       "Packager subcommands:\n"
       "pkgsend create-repository --set-property <section.property>=<value>\n"
       "pkgsend open [-en] pkg_fmri\n"
       "pkgsend add action arguments\n"
       "pkgsend import [-T file_pattern] bundlefile ...\n"
       "pkgsend include [-d basedir] .. [manifest] ...\n"
       "pkgsend close [-A | [--no-index] [--no-catalog]]\n"
       "pkgsend publish [ -d basedir] ... [--no-index] \n"
       "[--fmri-in-manifest | pkg_fmri] [--no-catalog] [manifest] ...\n"
       "pkgsend generate [-T file_pattern] bundlefile ....\n"
       "pkgsend refresh-index\n"
       "\n"
       "オプション:\n"
       "-s repo_uri target repository URI\n"
       "--help or -? display usage message\n"
       "\n"
       "環境:\n"
       "PKG_REPO"

#: ../publish.py:126
msgid ""
"Invalid repository configuration values were specified using --set-property "
"or required values are missing.  Please provide the correct and/or required "
"values using the --set-property option."
msgstr "--set-property を使用して、無効なリポジトリ構成値が指定されたか、必要な値が指定されていません。"
       "--set-property オプションを使用して、正しい値、または必要な値を指定してください。"

#: ../publish.py:149
msgid "only -e or -n may be specified"
msgstr "指定できるのは -e または -n だけです"

#: ../publish.py:152
msgid "open requires one package name"
msgstr "open ではパッケージ名を 1 つ指定する必要があります"

#: ../publish.py:183
msgid "No transaction ID specified using -t or in $PKG_TRANS_ID."
msgstr "-t の使用時、または $PKG_TRANS_ID でトランザクション ID が指定されていません。"

#: ../publish.py:198 ../publish.py:321 ../publish.py:401
msgid "No transaction ID specified in $PKG_TRANS_ID"
msgstr "$PKG_TRANS_ID にトランザクション ID が指定されていません"

#: ../publish.py:202 ../publish.py:413 ../publish.py:431
msgid "No arguments specified for subcommand."
msgstr "サブコマンドに引数が指定されていません。"

#: ../publish.py:230
msgid "No fmri argument specified for subcommand"
msgstr "サブコマンドに fmri の引数が指定されていません"

#: ../publish.py:275 ../publish.py:365
#, python-format
msgid "File %s line %s: %s"
msgstr "ファイル %s の行 %s: %s"

#: ../publish.py:281
msgid "Manifest does not set fmri and "
msgstr "マニフェストは FMRI を設定していません。"

#: ../publish.py:446
msgid "command does not take operands"
msgstr "コマンドはオペランドを取りません"

#: ../publish.py:475
#, python-format
msgid "pkgsend: illegal global option -- %s"
msgstr "pkgsend: 不正なグローバルオプション -- %s"

#: ../publish.py:511
#, python-format
msgid "illegal %s option -- %s"
msgstr "不正な %s オプション -- %s"

#: ../pull.py:86
msgid ""
"Usage:\n"
"        pkgrecv [-s src_repo_uri] [-d (path|dest_uri)] [-k] [-m] [-n] [-r]\n"
"            (fmri|pattern) ...\n"
"        pkgrecv [-s src_repo_uri] -n\n"
"\n"
"Options:\n"
"        -d path_or_uri  The path of a directory to save the retrieved "
"package\n"
"                        to, or the URI of a repository to republish it to.  "
"If\n"
"                        not provided, the default value is the current "
"working\n"
"                        directory.  If a directory path is provided, then\n"
"                        package content will only be retrieved if it does "
"not\n"
"                        already exist in the target directory.  If a "
"repository\n"
"                        URI is provided, a temporary directory will be "
"created\n"
"                        and all of the package data retrieved before "
"attempting\n"
"                        to republish it.\n"
"\n"
"        -h              Display this usage message.\n"
"        -k              Keep the retrieved package content compressed, "
"ignored\n"
"                        when republishing.  Should not be used with "
"pkgsend.\n"
"        -m match        Controls matching behaviour using the following "
"values:\n"
"                            all-timestamps\n"
"                                includes all matching timestamps, not just\n"
"                                latest (implies all-versions)\n"
"                            all-versions\n"
"                                includes all matching versions, not just "
"latest\n"
"        -n              List the most recent versions of the packages "
"available\n"
"                        from the specified repository and exit (all other\n"
"                        options except -s will be ignored).\n"
"        -r              Recursively evaluates all dependencies for the "
"provided\n"
"                        list of packages and adds them to the list.\n"
"        -s src_repo_uri A URI representing the location of a pkg(5)\n"
"                        repository to retrieve package data from.\n"
"\n"
"Environment:\n"
"        PKG_DEST        Destination directory or repository URI\n"
"        PKG_SRC         Source repository URI"
msgstr "使用方法:\n"
       "        pkgrecv [-s src_repo_uri] [-d (path|dest_uri)] [-k] [-m] [-n] [-r]\n"
       "            (fmri|pattern) ...\n"
       "        pkgrecv [-s src_repo_uri] -n\n"
       "\n"
       "オプション:\n"
       "        -d path_or_uri  取得したパッケージを保存するディレクトリのパス、\n"
       "                        または、リポジトリを再公開する URI。\n"
       "                        指定しない場合は、現在の作業用ディレクトリが\n"
       "                        デフォルト値となります。ディレクトリパスを指定した\n"
       "                        場合は、保存先のディレクトリにパッケージのコンテンツが\n"
       "                        存在しない場合のみ、取得されます。\n"
       "                        リポジトリの URI が指定された場合は、テンポラリ\n"
       "                        ディレクトリが作成され、再公開される前にすべての\n"
       "                        パッケージデータが取得されます。\n"
       "\n"
       "        -h              この使用方法メッセージを表示。\n"
       "        -k              パッケージコンテンツの圧縮を保持。\n"
       "                        再公開する場合は、無視されます。pkgsend と一緒に\n"
       "                        使用することはできません。"
       "        -m match        次の値を使用して、一致動作を制御します:\n"
       "                            all-timestamps\n"
       "                                最新のタイムスタンプだけではなく、一致するすべての\n"
       "                                タイムスタンプを全て含む (つまり、all-versions)\n"
       "                            all-versions\n"
       "                                最新のバージョンだけではなく、一致するすべての\n"
       "                                バージョンを含む\n"
       "        -n              指定したリポジトリから入手できる最新のパッケージを表示し、\n"
       "                        終了 (-s を除くすべてのオプションは無視されます)。\n"
       "        -r              指定したパッケージのリストの依存関係を再帰的に検証して、\n"
       "                        リストに追加。\n"
       "        -s src_repo_uri パッケージデータを取得する pkg(5) のリポジトリの場所を\n"
       "                        表す URI。\n"
       "\n"
       "環境:\n"
       "        PKG_DEST        保存先のディレクトリ、またはリポジトリ URI\n"
       "        PKG_SRC         取得元のリポジトリ URI"

#: ../pull.py:167
#, python-format
msgid "Unable to load manifest '%(mpath)s' for package '%(pfmri)s'."
msgstr "パッケージ '%(pfmri)s' のマニフェスト '%(mpath)s' を読み込めません。 "

#: ../pull.py:177
#, python-format
msgid "Unable to parse manifest '%(mpath)s' for package '%(pfmri)s'"
msgstr "パッケージ '%(pfmri)s' のマニフェスト '%(mpath)s' を解析できません。 "

#: ../pull.py:188
#, python-format
msgid "Unable to write manifest '%(mpath)s' for package '%(pfmri)s'."
msgstr "パッケージ '%(pfmri)s' のマニフェスト '%(mpath)s' を書き込めません。 "

#: ../pull.py:235
#, python-format
msgid "Unable to retrieve manifest %s from %s: %s"
msgstr "マニフェスト '%s' を '%s' から取得できません: %s"

#: ../pull.py:244
#, python-format
msgid "Error occurred while reading from: %s"
msgstr "%s からの読み取り中にエラーが発生しました"

#: ../pull.py:448
#, python-format
msgid "Unable to retrieve content from: %s"
msgstr "%s からコンテンツの取得ができません"

#: ../pull.py:480
#, python-format
msgid "Unable to extract file: %s"
msgstr "ファイル %s を抽出できません"

#: ../pull.py:527
#, python-format
msgid "Unable to download catalog from: %s"
msgstr "%s からカタログをダウンロードできません"

#: ../pull.py:538
#, python-format
msgid "Error: %s while reading from: %s"
msgstr "%$2s からの読み取り中にエラー %$1s が発生しました"

#: ../pull.py:576
#, python-format
msgid "Illegal option -- %s"
msgstr "不正なオプション -- %s"

#: ../pull.py:597
#, python-format
msgid "Illegal option value -- %s"
msgstr "不正なオプション値 -- %s"

#: ../pull.py:600
msgid "a source repository must be provided"
msgstr "ソースのリポジトリを指定する必要があります"

#: ../pull.py:605
msgid "-n takes no options"
msgstr "-n はオプションを取りません"

#: ../pull.py:612
msgid "must specify at least one pkgfmri"
msgstr "pkgfmri は少なくとも 1 つ指定する必要があります"

#: ../pull.py:636
msgid "To create a repository, use the pkgsend command."
msgstr "リポジトリを作成するには、pkgsend コマンドを使用してください。"

#: ../pull.py:641
#, python-format
msgid ""
"The repository configuration for the repository located at '%s' is not valid "
"or the specified path does not exist.  Please correct the configuration of "
"the repository or create a new one."
msgstr "'%s' にあるリポジトリの構成が無効か、指定したパスが存在しません。リポジトリの構成を修正するか、新しいリポジトリを作成してください。"

#: ../pull.py:656
#, python-format
msgid "Unable to create basedir '%s'."
msgstr "basedir '%s' を作成できません。"

#: ../pull.py:666
msgid "Retrieving manifests for dependency evaluation ..."
msgstr "依存関係の評価のためにマニフェストを取得しています..."

#: ../pull.py:684
msgid "Retrieving manifests for package evaluation ..."
msgstr "パッケージの評価のためにマニフェストを取得しています..."

#. Next, retrieve and store the content for each package.
#: ../pull.py:704
msgid "Retrieving package content ..."
msgstr "パッケージのコンテンツを取得しています..."

#: ../pull.py:726
#, python-format
msgid "Republishing %s ..."
msgstr "%s を再発行しています..."

#: ../pull.py:782
msgid "Refreshing repository search indices ..."
msgstr "リポジトリの検索インデックスを更新しています..."

#: ../um/data/updatemanager-preferences.schemas.in.h:1
msgid ""
"A text message describing the updates is displayed near the panel's "
"Notification Area when updates become available if set to true."
msgstr ""
       "true に設定された場合、更新が使用可能になった際に、その更新を"
       "説明するテキストメッセージがパネルの通知領域の近くに表示されます。"

#: ../um/data/updatemanager-preferences.schemas.in.h:2
msgid "How often to check for updates"
msgstr "更新のチェック頻度"

#: ../um/data/updatemanager-preferences.schemas.in.h:3
msgid ""
"How often to check for updates. The valid values are Daily, Weekly, Monthly, "
"Never"
msgstr ""
       "更新のチェック頻度を指定します。有効な値は「毎日」、"
       "「毎週」、「毎月」、「なし」です"

#: ../um/data/updatemanager-preferences.schemas.in.h:4
msgid "Start delay"
msgstr "開始遅延"

#: ../um/data/updatemanager-preferences.schemas.in.h:5
msgid "The notify icon is displayed on startup if set to true."
msgstr "true に設定された場合、起動時に通知アイコンが表示されます。"

#: ../um/data/updatemanager-preferences.schemas.in.h:6
msgid ""
"The time, in seconds, that the updatemanagernotifier should wait, after "
"starting, before checking for updates."
msgstr ""
       "開始したあとで更新をチェックする前に updatemanagernotifier が"
       "待機する秒数。"

#: ../um/data/updatemanager-preferences.schemas.in.h:7
msgid ""
"The updatemanagernotifier program terminates after the user activates the "
"icon in the notification area if set to true."
msgstr ""
       "true に設定された場合、ユーザーが通知領域のアイコンをアクティブにすると、"
       "updatemanagernotifier プログラムは終了します。"

#: ../um/data/updatemanager-preferences.schemas.in.h:8
msgid "Whether to always display the icon on startup"
msgstr "起動時に常にアイコンを表示するかどうか"

#: ../um/data/updatemanager-preferences.schemas.in.h:9
msgid "Whether to display text message when updates are available"
msgstr "更新が使用可能になった際にテキストメッセージを表示するかどうか"

#: ../um/data/updatemanager-preferences.schemas.in.h:10
msgid "Whether to terminate updatemanagernotiifer after icon activation."
msgstr "アイコンがアクティブになった際に updatemanagernotiifer を終了するかどうか。"

#: ../um/data/updatemanager.desktop.in.h:1
msgid "Install available updates"
msgstr "使用可能な更新をインストールします"

#: ../um/data/updatemanager.desktop.in.h:2 ../um/data/updatemanager.glade.h:11
#: ../updatemanager.py:563 ../updatemanagernotifier.py:329
msgid "Update Manager"
msgstr "更新マネージャー"

#: ../um/data/updatemanager.glade.h:1
msgid "<b>Evaluate</b>"
msgstr "<b>評価</b>"

#: ../um/data/updatemanager.glade.h:2 ../updatemanager.py:283
msgid "<b>Updates are available for the following packages</b>"
msgstr "<b>次のパッケージの更新が使用可能です</b>"

#: ../um/data/updatemanager.glade.h:3
msgid "<b>_Details</b>"
msgstr "<b>詳細(_D)</b>"

#: ../um/data/updatemanager.glade.h:5
msgid "Close dialog when _finished"
msgstr "終了時にダイアログを閉じる(_F)"

#: ../um/data/updatemanager.glade.h:7
msgid "Index"
msgstr "インデックス"

#: ../um/data/updatemanager.glade.h:8
msgid "Install Update"
msgstr "更新のインストール"

#: ../um/data/updatemanager.glade.h:10
msgid "Select the packages you want to update and click Install."
msgstr "更新するパッケージを選択し、「インストール」をクリックします。"

#: ../um/data/updatemanager.glade.h:12
msgid "_Cancel"
msgstr "キャンセル(_C)"

#: ../um/data/updatemanager.glade.h:14
msgid "_Install Updates"
msgstr "更新のインストール(_I)"

#: ../um/data/updatemanager.glade.h:15
msgid "_OK"
msgstr "了解(_O)"

#: ../um/data/updatemanager.glade.h:16
msgid "_Select all Updates"
msgstr "すべての更新を選択(_S)"

#: ../um/data/updatemanager.glade.h:17
msgid "_Update All"
msgstr "すべて更新(_U)"

#: ../um/data/updatemanagernotifier.desktop.in.h:1
msgid "Notify user when updates are available"
msgstr "更新が使用可能になった際にユーザーに通知します"

#: ../um/data/updatemanagernotifier.desktop.in.h:2
msgid "Update Manager Notifier"
msgstr "更新マネージャー通知"

#: ../updatemanager.py:214
#, python-format
msgid "GUI will not respond to any event! %s. Check updatemanager.py signals"
msgstr "GUI はどのイベントにも応答しません。%s。updatemanager.py のシグナルをチェックしてください"

#: ../updatemanager.py:285
msgid ""
"Click Update All to create a new boot environment and install all packages "
"into it."
msgstr "新しいブート環境を作成し、その中にすべてのパッケージをインストールするには、"
       "「すべて更新」をクリックします。"

#: ../updatemanager.py:312
msgid "Current Version"
msgstr "最新バージョン"

#: ../updatemanager.py:321
msgid "Size (Meg)"
msgstr "サイズ (M バイト)"

#: ../updatemanager.py:409
#, python-format
msgid "count: %d"
msgstr "カウント: %d"

#: ../updatemanager.py:429
msgid "<b>You do not have sufficient permissions</b>"
msgstr "<b>十分なアクセス権がありません<b>"

#: ../updatemanager.py:431
msgid "Please restart pm-updatemanager using pfexec."
msgstr "pfexec を使用して pm-updatemanager を再起動してください。"

#: ../updatemanager.py:435
msgid "<b>No Updates available</b>"
msgstr "<b>使用可能な更新はありません。</b>"

#: ../updatemanager.py:469
#, python-format
msgid ""
"\n"
"Fetching details for %s ..."
msgstr ""
       "\n"
       "%s の詳細の取得中..."

#: ../updatemanager.py:524
msgid ""
"\n"
"No details available"
msgstr ""
       "\n"
       "使用可能な詳細が存在しません"

#: ../updatemanager.py:582
msgid ""
"Not enough disc space, the Update All action cannot be performed.\n"
"\n"
"Click OK to manage your existing BEs and free up disk space or Cancel to "
"cancel Update All."
msgstr ""
       "ディスク容量が不足しているため「すべて更新」アクションを実行できません。\n"
       "\n"
       "既存の BE を管理してディスク容量を解放する場合は「了解」をクリックし、「すべて更新」を取り消す場合は"
       "「取消し」をクリックしてください。"

#: ../updatemanager.py:586
msgid "Not Enough Disc Space"
msgstr "ディスク容量が不足しています"

#: ../updatemanager.py:734
msgid "Checking for new software"
msgstr "新しいソフトウェアのチェック中"

#: ../updatemanagernotifier.py:308
msgid "Updates are available"
msgstr "更新が使用可能です"

#: ../updatemanagernotifier.py:330
msgid ""
"Updates available\n"
"Please click on icon to update."
msgstr ""
       "更新が使用可能です\n"
       "更新する場合はアイコンをクリックしてください。"

#: ../updatemanagernotifier.py:405
msgid "Another instance of UpdateManagerNotify is running"
msgstr "UpdateManagerNotify の別のインスタンスが実行中です"
